<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Canine Glioma Project 路 Extended Methods, Data, and Code for Figures"><meta name=author content="Samir B. Amin"><link href=https://canineglioma.verhaaklab.com/figures/A1_preload/ rel=canonical><link rel=icon href=../../assets/images/fixed/favicon.png><meta name=generator content="mkdocs-1.2.3, mkdocs-material-8.2.5"><title>Start Here &middot; Canine Glioma - Canine Glioma Project &middot; Verhaak Lab</title><link rel=stylesheet href=../../assets/stylesheets/main.2d9f7617.min.css><link rel=stylesheet href=../../assets/stylesheets/palette.e6a45f82.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CFira+Code:400,400i,700,700i&display=fallback"><style>:root{--md-text-font:"Roboto";--md-code-font:"Fira Code"}</style><link rel=stylesheet href=../../assets/css/extra.css><script>__md_scope=new URL("../..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script><!-- Determine analytics provider (deprecated, removed in v8) --><!-- Determine analytics provider --><!-- Set up analytics provider --><!-- Determine analytics property (deprecated, removed in v8) --><!-- Determine analytics property --><!-- Google Analytics 4 (G-XXXXXXXXXX) --><script>
    window.ga = window.ga || function() {
      (ga.q = ga.q || []).push(arguments)
    }
    ga.l = +new Date()

    /* Set up integration and send page view */
    ga("create", "UA-64053990-2", "auto")
    ga("set", "anonymizeIp", true)
    ga("send", "pageview")

    /* Register virtual event handlers */
    document.addEventListener("DOMContentLoaded", function() {
      if (document.forms.search) {
        var query = document.forms.search.query
        query.addEventListener("blur", function() {
          if (this.value) {
            var path = document.location.pathname;
            ga("send", "pageview", path + "?q=" + this.value)
          }
        })
      }

      /* Send page view on location change */
      if (typeof location$ !== "undefined")
        location$.subscribe(function(url) {
          ga("send", "pageview", url.pathname)
        })
    })
  </script><script async src=https://www.google-analytics.com/analytics.js></script><!--   <link rel="stylesheet" href="https://canineglioma.verhaaklab.com/../../assets/stylesheets/main.c296cc61.min.css"> --><meta name=keywords content="canine,glioma,pediatric,comparative genomics,DIPG,verhaak,lab,jax"><meta property=og:type content=website><meta property=og:title content="Start Here 路 Canine Glioma Project 路 Verhaak Lab"><meta property=og:description content="Canine Glioma Project 路 Extended Methods, Data, and Code for Figures"><meta content=https://canineglioma.verhaaklab.com/figures/A1_preload/ property=og:url><meta property=og:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta property=og:image:type content=image/png><meta property=og:image:width content=256><meta property=og:image:height content=256><meta name=twitter:card content=summary><meta name=twitter:site content=@sbamin><meta name=twitter:creator content=@sbamin><meta name=twitter:title content="Start Here 路 Canine Glioma Project 路 Verhaak Lab"><meta name=twitter:description content="Canine Glioma Project 路 Extended Methods, Data, and Code for Figures"><meta name=twitter:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta name=twitter:image:alt content="webpage banner or site logo"></head> <body dir=ltr data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo> <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#raw-sequence-data class=md-skip> Skip to content </a> </div> <div data-md-component=announce> </div> <div data-md-component=outdated hidden> <aside class="md-banner md-banner--warning"> </aside> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-header__button md-logo" aria-label="Canine Glioma Project 路 Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> Canine Glioma Project &middot; Verhaak Lab </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> Start Here &middot; Canine Glioma </span> </div> </div> </div> <form class=md-header__option data-md-component=palette> <input class=md-option data-md-color-media data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo aria-label="Switch to dark mode" type=radio name=__palette id=__palette_1> <label class="md-header__button md-icon" title="Switch to dark mode" for=__palette_2 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg> </label> <input class=md-option data-md-color-media data-md-color-scheme=slate data-md-color-primary=teal data-md-color-accent=light-green aria-label="Switch to light mode" type=radio name=__palette id=__palette_2> <label class="md-header__button md-icon" title="Switch to light mode" for=__palette_1 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3z"/></svg> </label> </form> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </label> <nav class=md-search__options aria-label=Search> <button type=reset class="md-search__icon md-icon" aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg> </button> </nav> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/sbamin/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 2A10 10 0 0 0 2 12c0 4.42 2.87 8.17 6.84 9.5.5.08.66-.23.66-.5v-1.69c-2.77.6-3.36-1.34-3.36-1.34-.46-1.16-1.11-1.47-1.11-1.47-.91-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.87 1.52 2.34 1.07 2.91.83.09-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.92 0-1.11.38-2 1.03-2.71-.1-.25-.45-1.29.1-2.64 0 0 .84-.27 2.75 1.02.79-.22 1.65-.33 2.5-.33.85 0 1.71.11 2.5.33 1.91-1.29 2.75-1.02 2.75-1.02.55 1.35.2 2.39.1 2.64.65.71 1.03 1.6 1.03 2.71 0 3.82-2.34 4.66-4.57 4.91.36.31.69.92.69 1.85V21c0 .27.16.59.67.5C19.14 20.16 22 16.42 22 12A10 10 0 0 0 12 2z"/></svg> </div> <div class=md-source__repository> sbamin/canineglioma </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class="md-tabs__inner md-grid"> <ul class=md-tabs__list> <li class=md-tabs__item> <a href=../.. class=md-tabs__link> Home </a> </li> <li class=md-tabs__item> <a href=../../methods/S01_samples/ class=md-tabs__link> Methods </a> </li> <li class=md-tabs__item> <a href=./ class="md-tabs__link md-tabs__link--active"> Figures </a> </li> <li class=md-tabs__item> <a href=../../run_code/ class=md-tabs__link> R Markdown </a> </li> <li class=md-tabs__item> <a href=https://verhaaklab.com/contact class=md-tabs__link> Contact </a> </li> </ul> </div> </nav> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-nav__button md-logo" aria-label="Canine Glioma Project 路 Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> Canine Glioma Project &middot; Verhaak Lab </label> <div class=md-nav__source> <a href=https://github.com/sbamin/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 2A10 10 0 0 0 2 12c0 4.42 2.87 8.17 6.84 9.5.5.08.66-.23.66-.5v-1.69c-2.77.6-3.36-1.34-3.36-1.34-.46-1.16-1.11-1.47-1.11-1.47-.91-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.87 1.52 2.34 1.07 2.91.83.09-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.92 0-1.11.38-2 1.03-2.71-.1-.25-.45-1.29.1-2.64 0 0 .84-.27 2.75 1.02.79-.22 1.65-.33 2.5-.33.85 0 1.71.11 2.5.33 1.91-1.29 2.75-1.02 2.75-1.02.55 1.35.2 2.39.1 2.64.65.71 1.03 1.6 1.03 2.71 0 3.82-2.34 4.66-4.57 4.91.36.31.69.92.69 1.85V21c0 .27.16.59.67.5C19.14 20.16 22 16.42 22 12A10 10 0 0 0 12 2z"/></svg> </div> <div class=md-source__repository> sbamin/canineglioma </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../.. class=md-nav__link> Home </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_2 type=checkbox id=__nav_2> <label class=md-nav__link for=__nav_2> Methods <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Methods data-md-level=1> <label class=md-nav__title for=__nav_2> <span class="md-nav__icon md-icon"></span> Methods </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../methods/S01_samples/ class=md-nav__link> Samples </a> </li> <li class=md-nav__item> <a href=../../methods/S02_data_sources/ class=md-nav__link> Data Sources </a> </li> <li class=md-nav__item> <a href=../../methods/S03_sample_prep/ class=md-nav__link> Sample Preparations </a> </li> <li class=md-nav__item> <a href=../../methods/S04_alignments_fprint/ class=md-nav__link> Alignments </a> </li> <li class=md-nav__item> <a href=../../methods/S05_somatic_variants/ class=md-nav__link> Somatic SNVs and Indels </a> </li> <li class=md-nav__item> <a href=../../methods/S06_smgs/ class=md-nav__link> Significantly Mutated Genes </a> </li> <li class=md-nav__item> <a href=../../methods/S07_hallmarks/ class=md-nav__link> Cancer Hallmarks </a> </li> <li class=md-nav__item> <a href=../../methods/S09_mutation_rate/ class=md-nav__link> Somatic Mutation Rate </a> </li> <li class=md-nav__item> <a href=../../methods/S10_SCNA/ class=md-nav__link> Somatic Copy Number Aberrations </a> </li> <li class=md-nav__item> <a href=../../methods/S11_aneuploidy/ class=md-nav__link> Aneuploidy Metrics </a> </li> <li class=md-nav__item> <a href=../../methods/S12_ith/ class=md-nav__link> Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../../methods/S13_mut_sigs/ class=md-nav__link> Mutational Signatures </a> </li> <li class=md-nav__item> <a href=../../methods/S14_mut_timing/ class=md-nav__link> Molecular Timing of Somatic Drivers </a> </li> <li class=md-nav__item> <a href=../../methods/S15_class_predict_methy/ class=md-nav__link> Class Prediction using methylation classifier </a> </li> <li class=md-nav__item> <a href=../../methods/S16_expression_immune/ class=md-nav__link> Immune Microenvironment Analysis </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3 type=checkbox id=__nav_3 checked> <label class=md-nav__link for=__nav_3> Figures <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Figures data-md-level=1> <label class=md-nav__title for=__nav_3> <span class="md-nav__icon md-icon"></span> Figures </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" data-md-toggle=toc type=checkbox id=__toc> <label class="md-nav__link md-nav__link--active" for=__toc> Start Here <span class="md-nav__icon md-icon"></span> </label> <a href=./ class="md-nav__link md-nav__link--active"> Start Here </a> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#raw-sequence-data class=md-nav__link> Raw sequence data </a> </li> <li class=md-nav__item> <a href=#r-session class=md-nav__link> R session </a> </li> <li class=md-nav__item> <a href=#load-r-packages class=md-nav__link> Load R packages </a> </li> <li class=md-nav__item> <a href=#load-r-objects class=md-nav__link> Load R objects </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_2 type=checkbox id=__nav_3_2> <label class=md-nav__link for=__nav_3_2> F1 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F1 data-md-level=2> <label class=md-nav__title for=__nav_3_2> <span class="md-nav__icon md-icon"></span> F1 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F1A/ class=md-nav__link> F1A &middot; Somatic Mutational Landscape </a> </li> <li class=md-nav__item> <a href=../F1B/ class=md-nav__link> F1B &middot; Somatic Recurrent Hotspot Mutations </a> </li> <li class=md-nav__item> <a href=../F1C/ class=md-nav__link> F1C &middot; Comparative Pathway Analysis </a> </li> <li class=md-nav__item> <a href=../F1D/ class=md-nav__link> F1D &middot; Comparative Somatic Mutational Burden </a> </li> <li class=md-nav__item> <a href=../S1/ class=md-nav__link> S1A &middot; DNA fingerprinting </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_3 type=checkbox id=__nav_3_3> <label class=md-nav__link for=__nav_3_3> F2 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F2 data-md-level=2> <label class=md-nav__title for=__nav_3_3> <span class="md-nav__icon md-icon"></span> F2 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F2A/ class=md-nav__link> F2A &middot; Somatic Copy Number Landscape </a> </li> <li class=md-nav__item> <a href=../F2B/ class=md-nav__link> F2B &middot; Comparative Aneuploidy Burden </a> </li> <li class=md-nav__item> <a href=../F2C/ class=md-nav__link> F2C &middot; Comparative Shared Syntenic Aneuploidy </a> </li> <li class=md-nav__item> <a href=../F2D/ class=md-nav__link> F2D &middot; Comparative Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../S2/ class=md-nav__link> S2 &middot; IGV Plots </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_4 type=checkbox id=__nav_3_4> <label class=md-nav__link for=__nav_3_4> F3 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F3 data-md-level=2> <label class=md-nav__title for=__nav_3_4> <span class="md-nav__icon md-icon"></span> F3 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F3A/ class=md-nav__link> F3A &middot; Somatic Mutational Signatures - Deconvoluted </a> </li> <li class=md-nav__item> <a href=../F3B/ class=md-nav__link> F3B &middot; Somatic Mutational Landscape - de novo </a> </li> <li class=md-nav__item> <a href=../F3C/ class=md-nav__link> F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes </a> </li> <li class=md-nav__item> <a href=../F3D/ class=md-nav__link> F3D &middot; Comparative Molecular Timing of Driver Events </a> </li> <li class=md-nav__item> <a href=../S3/ class=md-nav__link> S3 &middot; Suppl Figures </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_5 type=checkbox id=__nav_3_5> <label class=md-nav__link for=__nav_3_5> F4 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F4 data-md-level=2> <label class=md-nav__title for=__nav_3_5> <span class="md-nav__icon md-icon"></span> F4 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F4/ class=md-nav__link> F4 &middot; DNA Methylation based Prediction of Canine Gliomas </a> </li> <li class=md-nav__item> <a href=../S4/ class=md-nav__link> S4 &middot; Suppl Figure 4 </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_6 type=checkbox id=__nav_3_6> <label class=md-nav__link for=__nav_3_6> F5 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F5 data-md-level=2> <label class=md-nav__title for=__nav_3_6> <span class="md-nav__icon md-icon"></span> F5 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F5/ class=md-nav__link> F5B &middot; Comparative Immunohistochemistry - Quantitative </a> </li> <li class=md-nav__item> <a href=../S5/ class=md-nav__link> S5 &middot; Suppl Figure 5 </a> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_4 type=checkbox id=__nav_4> <div class="md-nav__link md-nav__link--index "> <a href=../../run_code/ >R Markdown</a> <label for=__nav_4> <span class="md-nav__icon md-icon"></span> </label> </div> <nav class=md-nav aria-label="R Markdown" data-md-level=1> <label class=md-nav__title for=__nav_4> <span class="md-nav__icon md-icon"></span> R Markdown </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../run_code/run_all/ class=md-nav__link> Output </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=https://verhaaklab.com/contact class=md-nav__link> Contact </a> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=sidebar data-md-type=toc> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#raw-sequence-data class=md-nav__link> Raw sequence data </a> </li> <li class=md-nav__item> <a href=#r-session class=md-nav__link> R session </a> </li> <li class=md-nav__item> <a href=#load-r-packages class=md-nav__link> Load R packages </a> </li> <li class=md-nav__item> <a href=#load-r-objects class=md-nav__link> Load R objects </a> </li> </ul> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <!-- Add copied contents from original content.html here --> <!-- https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/src/partials/content.html --> <!-- Edit button --> <!--
  Hack: check whether the content contains a h1 headline. If it
  doesn't, the page title (or respectively site name) is used
  as the main headline.
--> <h1>Start Here</h1> <!-- Markdown content --> <h3 id=raw-sequence-data>Raw sequence data<a class=headerlink href=#raw-sequence-data title="Permanent link">&para;</a></h3> <p>Sequencing data generated from canine patients with glioma (n=83 donors) is available in the Binary Alignment Map (BAM) format (n=373) at the NCBI SRA database under BioProject ID: <a href=https://www.ncbi.nlm.nih.gov/bioproject/PRJNA579792>PRJNA579792</a>.</p> <p><a href=/methods/S02_data_sources/ >View details</a> on comparative sequencing data used from patients of human pediatric and adult glioma.</p> <h3 id=r-session>R session<a class=headerlink href=#r-session title="Permanent link">&para;</a></h3> <ul class=task-list> <li class=task-list-item><label class=task-list-control><input type=checkbox checked><span class=task-list-indicator></span></label> Prefer using RStudio or similar GUI over copy-pasting code in terminal. The latter may emit errors due to issues with syntax highlighting and font ligatures used on this website.</li> <li class=task-list-item><label class=task-list-control><input type=checkbox checked><span class=task-list-indicator></span></label> Code on this website was evaluated in an isolated, new environment and syntax-related errors were addressed on the top of respective code blocks. Please <a href=https://github.com/TheJacksonLaboratory/canineglioma/issues>report bugs</a> by submitting a GitHub issue. </li> </ul> <div class=highlight><pre><span></span><code><span class=nf>sessionInfo</span><span class=p>()</span>
</code></pre></div> <blockquote> <p>Package versions may have updated versions here than what was used at the time of publication. Versions for core R packages that were used in the manuscript are listed in the manuscript under STAR methods <code>=&gt;</code> Key Resources Table.</p> </blockquote> <div class=highlight><pre><span></span><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-conda_cos6-linux-gnu (64-bit)
Running under: CentOS release 6.5 (Final)

Matrix products: default
BLAS/LAPACK: /home/foo/anaconda3/lib/libopenblasp-r0.3.6.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] GenomicFeatures_1.36.4    AnnotationDbi_1.46.0     
 [3] MutationalPatterns_1.10.0 rtracklayer_1.44.0       
 [5] cowplot_1.0.0             ggrepel_0.8.1            
 [7] ggforce_0.3.1             gridExtra_2.3            
 [9] forcats_0.4.0             stringr_1.4.0            
[11] dplyr_0.8.3               purrr_0.3.3              
[13] readr_1.3.1               tidyr_1.0.2              
[15] tibble_2.1.3              ggplot2_3.2.1            
[17] tidyverse_1.2.1           Palimpsest_1.0.0         
[19] GenomicRanges_1.36.0      GenomeInfoDb_1.20.0      
[21] IRanges_2.18.1            S4Vectors_0.22.0         
[23] NMF_0.21.0                Biobase_2.44.0           
[25] BiocGenerics_0.30.0       cluster_2.1.0            
[27] rngtools_1.4              pkgmaker_0.27            
[29] registry_0.5-1            extrafont_0.17          

loaded via a namespace (and not attached):
  [1] colorspace_1.4-1            ggsignif_0.5.0             
  [3] ellipsis_0.3.0              rprojroot_1.3-2            
  [5] IRdisplay_0.7.0             lsa_0.73.1                 
  [7] XVector_0.24.0              ggdendro_0.1-20            
  [9] base64enc_0.1-3             rstudioapi_0.10            
 [11] ggpubr_0.2.1.999            farver_1.1.0               
 [13] SnowballC_0.6.0             bit64_0.9-7                
 [15] fansi_0.4.0                 lubridate_1.7.4            
 [17] xml2_1.2.2                  codetools_0.2-16           
 [19] doParallel_1.0.14           polyclip_1.10-0            
 [21] IRkernel_1.0.2              jsonlite_1.6               
 [23] Rsamtools_2.0.0             broom_0.5.4                
 [25] gridBase_0.4-7              graph_1.62.0               
 [27] compiler_3.6.1              httr_1.4.1                 
 [29] backports_1.1.5             assertthat_0.2.1           
 [31] Matrix_1.2-17               lazyeval_0.2.2             
 [33] cli_2.0.1                   tweenr_1.0.1               
 [35] htmltools_0.3.6             prettyunits_1.0.2          
 [37] tools_3.6.1                 gtable_0.3.0               
 [39] glue_1.3.1                  GenomeInfoDbData_1.2.1     
 [41] reshape2_1.4.3              Rcpp_1.0.3                 
 [43] cellranger_1.1.0            vctrs_0.2.2                
 [45] Biostrings_2.52.0           gdata_2.18.0               
 [47] nlme_3.1-140                iterators_1.0.10           
 [49] rvest_0.3.5                 lifecycle_0.1.0            
 [51] gtools_3.8.1                XML_3.98-1.20              
 [53] MASS_7.3-51.4               zlibbioc_1.30.0            
 [55] RCircos_1.2.1               scales_1.0.0               
 [57] BSgenome_1.52.0             VariantAnnotation_1.30.1   
 [59] hms_0.5.3                   SummarizedExperiment_1.14.0
 [61] RColorBrewer_1.1-2          memoise_1.1.0              
 [63] biomaRt_2.40.1              stringi_1.4.3              
 [65] RSQLite_2.1.1               foreach_1.4.4              
 [67] plotrix_3.7-6               caTools_1.17.1.2           
 [69] BiocParallel_1.18.0         bibtex_0.4.2               
 [71] repr_1.0.1                  rlang_0.4.3                
 [73] pkgconfig_2.0.3             matrixStats_0.54.0         
 [75] bitops_1.0-6                pracma_2.2.5               
 [77] evaluate_0.14               lattice_0.20-38            
 [79] labeling_0.3                GenomicAlignments_1.20.1   
 [81] bit_1.1-14                  tidyselect_0.2.5           
 [83] here_0.1                    plyr_1.8.4                 
 [85] magrittr_1.5                R6_2.4.1                   
 [87] gplots_3.0.1.1              generics_0.0.2             
 [89] pbdZMQ_0.3-3                DelayedArray_0.10.0        
 [91] DBI_1.1.0                   pillar_1.4.3               
 [93] haven_2.2.0                 withr_2.1.2                
 [95] RCurl_1.95-4.12             modelr_0.1.5               
 [97] crayon_1.3.4                uuid_0.1-2                 
 [99] KernSmooth_2.23-15          progress_1.2.2             
[101] grid_3.6.1                  readxl_1.3.1               
[103] data.table_1.12.2           blob_1.2.0                 
[105] Rgraphviz_2.28.0            digest_0.6.23              
[107] xtable_1.8-4                munsell_0.5.0              
</code></pre></div> <h3 id=load-r-packages>Load R packages<a class=headerlink href=#load-r-packages title="Permanent link">&para;</a></h3> <blockquote> <p>PS: Not all of R packages were loaded for generating all figures. If you are loading all of packages, be aware of warnings related to masked functions and use pacakge name prefix, e.g., <code>dplyr::select</code> to explicitly call a function.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>## For Figure 1 and 2, including supplemental figures</span>
<span class=nf>library</span><span class=p>(</span><span class=n>knitr</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>htmlTable</span><span class=p>)</span>

<span class=nf>library</span><span class=p>(</span><span class=n>tidyverse</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>broom</span><span class=p>)</span>

<span class=nf>library</span><span class=p>(</span><span class=n>maftools</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>tint</span><span class=p>)</span>

<span class=nf>library</span><span class=p>(</span><span class=n>DBI</span><span class=p>)</span>

<span class=nf>library</span><span class=p>(</span><span class=n>ggplot2</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>gridExtra</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>ggforce</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>ggrepel</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>cowplot</span><span class=p>)</span>

<span class=c1>## For Figure 3, including supplemental figures,</span>
<span class=c1>## load these extra packages.</span>

<span class=nf>library</span><span class=p>(</span><span class=n>Palimpsest</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>BSgenome.Cfamiliaris.UCSC.canFam3</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>rtracklayer</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>MutationalPatterns</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>tidyverse</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>GenomicFeatures</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>NMF</span><span class=p>)</span>
<span class=nf>library</span><span class=p>(</span><span class=n>BradleyTerryScalable</span><span class=p>)</span>

<span class=c1>## Import Arial fonts to avoid error</span>
<span class=c1>## related to font not found</span>
<span class=nf>library</span><span class=p>(</span><span class=n>extrafont</span><span class=p>)</span>
<span class=nf>loadfonts</span><span class=p>()</span>
</code></pre></div> <h3 id=load-r-objects>Load R objects<a class=headerlink href=#load-r-objects title="Permanent link">&para;</a></h3> <p><img alt= class=twemoji src=https://twemoji.maxcdn.com/v/latest/svg/1f680.svg title=:rocket:> Download <a href=/data/cgp_base_objects_v1_20200207.RData>/data/cgp_base_objects_v1_20200207.RData</a>, md5sum: 12812097f49d8c8300a470491387c7d1.</p> <div class=highlight><pre><span></span><code><span class=nf>load</span><span class=p>(</span><span class=s>&quot;cgp_base_objects_v1_20200207.RData&quot;</span><span class=p>)</span>
<span class=nf>ls</span><span class=p>()</span>
<span class=c1>## 37 objects</span>
</code></pre></div> <table> <thead> <tr> <th>R object</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td>cgp_suppl_sample_info_master</td> <td>Donor level information on 81 canine patients with glioma.</td> </tr> <tr> <td>maf_nogistic</td> <td>maftools compatible MAF object, does not contain copy number data on 81 canine patients. Note that three patients had no detectable somatic mutations following somatic mutation filtration.</td> </tr> <tr> <td>cgp_maftools_gistic_matched</td> <td>maftools compatible MAF object <strong>only</strong> on 67 canine patients with paired tumor-normal samples where somatic copy number data was available.</td> </tr> <tr> <td>cgp_maftools_gistic_n81</td> <td>maftools compatible MAF object of 81 canine patients. This object <strong>also</strong> contain copy number data on 67 of 81 canine patients where paired tumor-normal samples were available.</td> </tr> <tr> <td>cgp_maftools_gistic_methy</td> <td>maftools compatible MAF object of 42 canine patients where DNA methylation data was available.</td> </tr> <tr> <td>cancer_genes_maftools</td> <td>Cancer genes with observed somatic mutation in canine patients (n=48). Based on cancer hallmark analysis (Fig 1B).</td> </tr> <tr> <td>HALLMARK</td> <td>Genes (n=277) to Cancer Hallmarks (n=11) mapping</td> </tr> <tr> <td>CG_all</td> <td>Somatic variants in canine glioma cohort (n=62/81 cases)</td> </tr> <tr> <td>ALL_HALLMARK_v1</td> <td>Cancer Hallmark table for Figure 1C</td> </tr> <tr> <td>mutrate_table</td> <td>Somatic mutation burden across patients of canine and human pediatric and adult cancers (n=4,840).</td> </tr> <tr> <td>merged_anp_scores</td> <td>Comparative aneuploidy metrics showing fraction of genome with aneuploidy</td> </tr> <tr> <td>cgp_aneuploidy_metrics_master</td> <td>aneuploidy metrics for 67 canine glioma patients.</td> </tr> <tr> <td>mat_aneuploidy_summary_mostvar_cgAll</td> <td>Comparative shared syntenic aneuploidy across canine and human pediatric and adult glioma.</td> </tr> <tr> <td>updated_merged_entropy_df_ridges</td> <td>Shannon entropy across canine and human pediatric and adult glioma where whole-genome sequence data was available (n=105).</td> </tr> <tr> <td>merged_ccf_vaf_df</td> <td>Comparative intra-tumor heterogeneity (Shannon entropy) and cellular prevalence matrix.</td> </tr> <tr> <td>mut_mat</td> <td>Canine glioma mutational matrix based on 96-trinucleotide context - Derived using <code>mut_matrix</code> function in <em>MutationalPatterns</em> R package.</td> </tr> <tr> <td>cancer_signatures</td> <td>Known human mutational signatures in adult (n=30, COSMIC v2) and pediatric cancers (n=12).</td> </tr> <tr> <td>fit_res_hm</td> <td>Somatic mutational signatures in canine glioma patients with outlier mutation profile (n=8). Signatures were fitted to known human mutational signatures.</td> </tr> <tr> <td>fit_res_nonhm</td> <td>Somatic mutational signatures in canine glioma patients without outlier mutation profile (n=73). Signatures were fitted to known human mutational signatures.</td> </tr> <tr> <td>hm_nonhm_topsigs_tbl, driver_sigs_groups</td> <td>Mutational signatures and their grouping to proposed mechnaism</td> </tr> <tr> <td>instability_sigs_df</td> <td>subset of <code>fit_res$contribution</code> matrix for suppl fig 3B.</td> </tr> <tr> <td>merged_ag_pg_cg_denovo_sig</td> <td>De-novo mutational signatures from canine, human pediatric and adult cohort.</td> </tr> <tr> <td>trimmed_{cg,pg,ag}_vcf</td> <td>Trimmed VCFs for plotting Figure 3C</td> </tr> <tr> <td>win_table_{cg,pg,ag}_vcf</td> <td>Trimmed VCFs for plotting Figure 3D</td> </tr> <tr> <td>cgp_RRBS.noNormals_class_prediction</td> <td>class prediction table for Figure 4</td> </tr> <tr> <td>cgp_RRBS.noNormals_sample_info</td> <td>Donor level information for 45 canine patients with glioma where DNA methylation data was available.</td> </tr> <tr> <td>{cgp, tcga}_info</td> <td>Sample metadata for suppl figure 5.</td> </tr> <tr> <td>{cgp, ped, tcga}_cibersort</td> <td>CIBERSORT output for three cohorts.</td> </tr> <tr> <td>fig5b_dat</td> <td>Data related to IHC panel, Figure 5B.</td> </tr> </tbody> </table> <p>We now have also updated CanFam3.1 aligned RNA-seq data<sup id=fnref:methods><a class=footnote-ref href=#fn:methods>1</a></sup> as tidyverse-compliant tibble format.</p> <ul> <li>Download <a href=/data/cgp_rnaseq_sleuth_tpm_normed_canfam3.1_v201908.rds>/data/cgp_rnaseq_sleuth_tpm_normed_canfam3.1_v201908.rds</a>, md5sum: b6159d234566a290e2a4105fa62b2cdb.</li> <li>Load data as follows:</li> </ul> <div class=highlight><pre><span></span><code>library<span class=o>(</span>tidyverse<span class=o>)</span>
rnaseq &lt;- readRDS<span class=o>(</span><span class=s2>&quot;cgp_rnaseq_sleuth_tpm_normed_canfam3.1_v201908.rds&quot;</span><span class=o>)</span>
glimpse<span class=o>(</span>rnaseq<span class=o>)</span>
</code></pre></div> <div class=footnote> <hr> <ol> <li id=fn:methods> <p><a href=/methods/S04_alignments_fprint/#rna-alignments>Details on RNA-seq alignments</a>&#160;<a class=footnote-backref href=#fnref:methods title="Jump back to footnote 1 in the text">&#8617;</a></p> </li> </ol> </div> <!-- Source file information --> <hr> <div class=md-source-file> <small> Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-iso_date">2020-08-27</span> </small> </div> <!-- end default contents --> <!-- Get setting from mkdocs.yml, but allow page-level overrides --> <!-- Inject Disqus into current page --> <p><small>Author: <a href=https://sbamin.com/about rel=me>Samir B. Amin</a> <span class=twemoji> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg></span><a href=https://twitter.com/sbamin>@sbamin</a></small></p> <!-- Giscus --> <h2 id=__comments>Comments</h2> <p>Comments are powered by <span class=twemoji><svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 16 16"><path fill-rule=evenodd d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg></span> <a href=https://github.com/TheJacksonLaboratory/canineglioma/discussions>github discussions</a></p> <!-- Replace with generated snippet --> <script src=https://giscus.app/client.js data-repo=TheJacksonLaboratory/canineglioma data-repo-id="MDEwOlJlcG9zaXRvcnkxOTEyOTczMjg=" data-category=webchat data-category-id=DIC_kwDOC2b3MM4CBGtB data-mapping=pathname data-reactions-enabled=1 data-emit-metadata=0 data-input-position=top data-theme=light data-lang=en crossorigin=anonymous async>
	</script> <!-- Reload on palette change --> <script>
	var palette = __md_get("__palette")
	if (palette && typeof palette.color === "object")
	  if (palette.color.scheme === "slate") {
	    var giscus = document.querySelector("script[src*=giscus]")
	    giscus.setAttribute("data-theme", "dark") 
	  }

	/* Register event handlers after documented loaded */
	document.addEventListener("DOMContentLoaded", function() {
	  var ref = document.querySelector("[data-md-component=palette]")
	  ref.addEventListener("change", function() {
	    var palette = __md_get("__palette")
	    if (palette && typeof palette.color === "object") {
	      var theme = palette.color.scheme === "slate" ? "dark" : "light"

	      /* Instruct Giscus to change theme */
	      var frame = document.querySelector(".giscus-frame")
	      frame.contentWindow.postMessage(
	        { giscus: { setConfig: { theme } } },
	        "https://giscus.app"
	      )
	    }
	  })
	})
	</script> </article> </div> </div> <a href=# class="md-top md-icon" data-md-component=top data-md-state=hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12z"/></svg> Back to top </a> </main> <!-- Footer --> <footer class=md-footer> <!-- Link to previous and/or next page --> <nav class="md-footer__inner md-grid" aria-label=Footer> <!-- Link to previous page --> <a href=../../methods/S16_expression_immune/ class="md-footer__link md-footer__link--prev" aria-label="Previous: Immune Microenvironment Analysis" rel=prev> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </div> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Previous </span> Immune Microenvironment Analysis </div> </div> </a> <!-- Link to next page --> <a href=../F1A/ class="md-footer__link md-footer__link--next" aria-label="Next: F1A &middot; Somatic Mutational Landscape" rel=next> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Next </span> F1A &middot; Somatic Mutational Landscape </div> </div> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg> </div> </a> </nav> <!-- Further information --> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <!-- Copyright and theme information --> <div class=md-copyright> <div class=md-copyright__highlight> &copy; 2010-2023: <a href=https://verhaaklab.com title=Home>Verhaak Lab</a> | <a href=https://verhaaklab.com/privacy/ >Disclosure</a> </div> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> <!-- Social links --> <div class=md-social> <a href=https://verhaaklab.com/contact target=_blank rel=noopener title=verhaaklab.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M448 64H64C28.65 64 0 92.65 0 128v256c0 35.35 28.65 64 64 64h384c35.35 0 64-28.65 64-64V128c0-35.35-28.7-64-64-64zM64 112h384c8.822 0 16 7.178 16 16v22.16l-166.8 138.1c-23.19 19.28-59.34 19.27-82.47.016L48 150.2V128c0-8.8 7.18-16 16-16zm384 288H64c-8.822 0-16-7.178-16-16V212.7l136.1 113.4c20.2 16.7 45.7 25.9 71.9 25.9s51.75-9.188 71.97-25.98L464 212.7V384c0 8.8-7.2 16-16 16z"/></svg> </a> <a href=https://twitter.com/roelverhaak target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> <a href=https://github.com/sbamin target=_blank rel=noopener title=github.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 496 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg> </a> <a href=https://twitter.com/sbamin target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "../..", "features": ["navigation.tabs", "navigation.instant", "navigation.top", "navigation.indexes", "navigation.sections", "content.code.annotate"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../assets/javascripts/workers/search.bd0b6b67.min.js", "version": {"provider": "mike", "default": "latest"}}</script> <script src=../../assets/javascripts/bundle.467223ff.min.js></script> <script src=../../assets/js/extra.js></script> <script src=../../assets/js/mathjax.js></script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js></script> <script src=https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js></script> <script src=../../assets/js/tables.js></script> </body> </html>