<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=keywords content="canine,glioma,pediatric,comparative genomics,DIPG,verhaak,lab,jax"><meta name=author content="Samir B. Amin"><link href=https://canineglioma.verhaaklab.com/figures/F3D/ rel=canonical><link rel=icon href=/assets/images/logo.png><meta name=generator content="mkdocs-1.2.3, mkdocs-material-7.3.6"><title>F3D &middot; Comparative Molecular Timing of Driver Events - Canine Glioma Project &middot; Verhaak Lab</title><link rel=stylesheet href=../../assets/stylesheets/main.a57b2b03.min.css><link rel=stylesheet href=../../assets/stylesheets/palette.3f5d1f46.min.css><meta name=theme-color content=#2094f3><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:300,400,400i,700%7CFira+Code&display=fallback"><style>:root{--md-text-font-family:"IBM Plex Sans";--md-code-font-family:"Fira Code"}</style><link rel=stylesheet href=/assets/css/extra.css><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.css><script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-64053990-2","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){if(this.value){var e=document.location.pathname;ga("send","pageview",e+"?q="+this.value)}})}),document.addEventListener("DOMContentSwitch",function(){ga("send","pageview",document.location.pathname)})</script><script async src=https://www.google-analytics.com/analytics.js></script><meta property=og:type content=website><meta property=og:title content="Canine Glioma Project · Verhaak Lab - F3D · Comparative Molecular Timing of Driver Events"><meta property=og:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta content=https://canineglioma.verhaaklab.com/figures/F3D/ property=og:url><meta property=og:image content=https://canineglioma.verhaaklab.com//assets/images/logo.png><meta property=og:image:type content=image/png><meta property=og:image:width content=256><meta property=og:image:height content=256><meta name=twitter:card content=summary><meta name=twitter:site content=@sbamin><meta name=twitter:creator content=@sbamin><meta name=twitter:title content="Canine Glioma Project · Verhaak Lab - F3D · Comparative Molecular Timing of Driver Events"><meta name=twitter:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=twitter:image content=https://canineglioma.verhaaklab.com//assets/images/logo.png><link rel=stylesheet href=/assets/stylesheets/overrides.2b41d529.min.css></head> <body dir=ltr data-md-color-scheme data-md-color-primary=blue data-md-color-accent=indigo> <script>function __prefix(e){return new URL("../..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#winning-tables class=md-skip> Skip to content </a> </div> <div data-md-component=announce> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-header__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/logo.png alt=logo> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> Canine Glioma Project &middot; Verhaak Lab </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> F3D &middot; Comparative Molecular Timing of Driver Events </span> </div> </div> </div> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </label> <nav class=md-search__options aria-label=Search> <a href=javascript:void(0) class="md-search__icon md-icon" aria-label=Share data-clipboard data-clipboard-text data-md-component=search-share tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08z"/></svg> </a> <button type=reset class="md-search__icon md-icon" aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg> </button> </nav> <div class=md-search__suggest data-md-component=search-suggest></div> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma/ title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-nav__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/logo.png alt=logo> </a> Canine Glioma Project &middot; Verhaak Lab </label> <div class=md-nav__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma/ title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../.. class=md-nav__link> Home </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_2 type=checkbox id=__nav_2> <label class=md-nav__link for=__nav_2> Methods <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Methods data-md-level=1> <label class=md-nav__title for=__nav_2> <span class="md-nav__icon md-icon"></span> Methods </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../methods/S01_samples/ class=md-nav__link> Samples </a> </li> <li class=md-nav__item> <a href=../../methods/S02_data_sources/ class=md-nav__link> Data Sources </a> </li> <li class=md-nav__item> <a href=../../methods/S03_sample_prep/ class=md-nav__link> Sample Preparations </a> </li> <li class=md-nav__item> <a href=../../methods/S04_alignments_fprint/ class=md-nav__link> Alignments </a> </li> <li class=md-nav__item> <a href=../../methods/S05_somatic_variants/ class=md-nav__link> Somatic SNVs and Indels </a> </li> <li class=md-nav__item> <a href=../../methods/S06_smgs/ class=md-nav__link> Significantly Mutated Genes </a> </li> <li class=md-nav__item> <a href=../../methods/S07_hallmarks/ class=md-nav__link> Cancer Hallmarks </a> </li> <li class=md-nav__item> <a href=../../methods/S09_mutation_rate/ class=md-nav__link> Somatic Mutation Rate </a> </li> <li class=md-nav__item> <a href=../../methods/S10_SCNA/ class=md-nav__link> Somatic Copy Number Aberrations </a> </li> <li class=md-nav__item> <a href=../../methods/S11_aneuploidy/ class=md-nav__link> Aneuploidy Metrics </a> </li> <li class=md-nav__item> <a href=../../methods/S12_ith/ class=md-nav__link> Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../../methods/S13_mut_sigs/ class=md-nav__link> Mutational Signatures </a> </li> <li class=md-nav__item> <a href=../../methods/S14_mut_timing/ class=md-nav__link> Molecular Timing of Somatic Drivers </a> </li> <li class=md-nav__item> <a href=../../methods/S15_class_predict_methy/ class=md-nav__link> Class Prediction using methylation classifier </a> </li> <li class=md-nav__item> <a href=../../methods/S16_expression_immune/ class=md-nav__link> Immune Microenvironment Analysis </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3 type=checkbox id=__nav_3 checked> <label class=md-nav__link for=__nav_3> Figures <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Figures data-md-level=1> <label class=md-nav__title for=__nav_3> <span class="md-nav__icon md-icon"></span> Figures </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../A1_preload/ class=md-nav__link> Start Here </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_2 type=checkbox id=__nav_3_2> <label class=md-nav__link for=__nav_3_2> F1 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F1 data-md-level=2> <label class=md-nav__title for=__nav_3_2> <span class="md-nav__icon md-icon"></span> F1 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F1A/ class=md-nav__link> F1A &middot; Somatic Mutational Landscape </a> </li> <li class=md-nav__item> <a href=../F1B/ class=md-nav__link> F1B &middot; Somatic Recurrent Hotspot Mutations </a> </li> <li class=md-nav__item> <a href=../F1C/ class=md-nav__link> F1C &middot; Comparative Pathway Analysis </a> </li> <li class=md-nav__item> <a href=../F1D/ class=md-nav__link> F1D &middot; Comparative Somatic Mutational Burden </a> </li> <li class=md-nav__item> <a href=../S1/ class=md-nav__link> S1A &middot; DNA fingerprinting </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_3 type=checkbox id=__nav_3_3> <label class=md-nav__link for=__nav_3_3> F2 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F2 data-md-level=2> <label class=md-nav__title for=__nav_3_3> <span class="md-nav__icon md-icon"></span> F2 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F2A/ class=md-nav__link> F2A &middot; Somatic Copy Number Landscape </a> </li> <li class=md-nav__item> <a href=../F2B/ class=md-nav__link> F2B &middot; Comparative Aneuploidy Burden </a> </li> <li class=md-nav__item> <a href=../F2C/ class=md-nav__link> F2C &middot; Comparative Shared Syntenic Aneuploidy </a> </li> <li class=md-nav__item> <a href=../F2D/ class=md-nav__link> F2D &middot; Comparative Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../S2/ class=md-nav__link> S2 &middot; IGV Plots </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_4 type=checkbox id=__nav_3_4 checked> <label class=md-nav__link for=__nav_3_4> F3 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F3 data-md-level=2> <label class=md-nav__title for=__nav_3_4> <span class="md-nav__icon md-icon"></span> F3 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F3A/ class=md-nav__link> F3A &middot; Somatic Mutational Signatures - Deconvoluted </a> </li> <li class=md-nav__item> <a href=../F3B/ class=md-nav__link> F3B &middot; Somatic Mutational Landscape - de novo </a> </li> <li class=md-nav__item> <a href=../F3C/ class=md-nav__link> F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes </a> </li> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" data-md-toggle=toc type=checkbox id=__toc> <label class="md-nav__link md-nav__link--active" for=__toc> F3D &middot; Comparative Molecular Timing of Driver Events <span class="md-nav__icon md-icon"></span> </label> <a href=./ class="md-nav__link md-nav__link--active"> F3D &middot; Comparative Molecular Timing of Driver Events </a> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#winning-tables class=md-nav__link> Winning Tables </a> </li> <li class=md-nav__item> <a href=#fig-3d class=md-nav__link> Fig 3D </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=../S3/ class=md-nav__link> S3 &middot; Suppl Figures </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_5 type=checkbox id=__nav_3_5> <label class=md-nav__link for=__nav_3_5> F4 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F4 data-md-level=2> <label class=md-nav__title for=__nav_3_5> <span class="md-nav__icon md-icon"></span> F4 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F4/ class=md-nav__link> F4 &middot; DNA Methylation based Prediction of Canine Gliomas </a> </li> <li class=md-nav__item> <a href=../S4/ class=md-nav__link> S4 &middot; Suppl Figure 4 </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_6 type=checkbox id=__nav_3_6> <label class=md-nav__link for=__nav_3_6> F5 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F5 data-md-level=2> <label class=md-nav__title for=__nav_3_6> <span class="md-nav__icon md-icon"></span> F5 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F5/ class=md-nav__link> F5B &middot; Comparative Immunohistochemistry - Quantitative </a> </li> <li class=md-nav__item> <a href=../S5/ class=md-nav__link> S5 &middot; Suppl Figure 5 </a> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_4 type=checkbox id=__nav_4> <label class=md-nav__link for=__nav_4> R Markdown <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="R Markdown" data-md-level=1> <label class=md-nav__title for=__nav_4> <span class="md-nav__icon md-icon"></span> R Markdown </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../run_code/ class=md-nav__link> How-To </a> </li> <li class=md-nav__item> <a href=../../run_code/run_all/ class=md-nav__link> Output </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=https://verhaaklab.com/contact class=md-nav__link> Contact </a> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=sidebar data-md-type=toc> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#winning-tables class=md-nav__link> Winning Tables </a> </li> <li class=md-nav__item> <a href=#fig-3d class=md-nav__link> Fig 3D </a> </li> </ul> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <a href=https://github.com/TheJacksonLaboratory/canineglioma/edit/master/docs/figures/F3D.md title="Edit this page" class="md-content__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"/></svg> </a> <h1>F3D &middot; Comparative Molecular Timing of Driver Events</h1> <h3 id=winning-tables>Winning Tables<a class=headerlink href=#winning-tables title="Permanent link">&para;</a></h3> <blockquote> <p>Make winning matrix of somatic variants in cancer driver genes (n=73) based on inferred clonality using <a href=https://github.com/FunGeST/Palimpsest><em>Palimpsest</em></a> R package<sup id=fnref:1><a class=footnote-ref href=#fn:1>1</a></sup>. Author of Palimpsest package has provided <a href=https://github.com/FunGeST/Palimpsest/raw/master/Files/vignette_palimpsest_2.0.pdf>a detailed documentation</a> on how to run Palimpsest workflow.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>## function to trim vcfs and make winning tables</span>

<span class=n>mk_win_table</span> <span class=o>&lt;-</span> <span class=nf>function</span><span class=p>(</span><span class=n>win_table_vcf</span><span class=p>)</span> <span class=p>{</span>

  <span class=n>vcf_timing</span> <span class=o>&lt;-</span> <span class=n>win_table_vcf</span> <span class=o>%&gt;%</span>
    <span class=nf>add_count</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span> <span class=n>timing_state</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>result</span> <span class=o>=</span> <span class=nf>case_when</span><span class=p>(</span><span class=nf>grepl</span><span class=p>(</span><span class=s>&quot;early_clonal&quot;</span><span class=p>,</span> <span class=n>timing_state</span><span class=p>)</span> <span class=o>~</span> <span class=m>5</span><span class=p>,</span>
                              <span class=nf>grepl</span><span class=p>(</span><span class=s>&quot;early_subclonal&quot;</span><span class=p>,</span> <span class=n>timing_state</span><span class=p>)</span> <span class=o>~</span> <span class=m>1</span><span class=p>,</span>
                              <span class=nf>grepl</span><span class=p>(</span><span class=s>&quot;late_clonal&quot;</span><span class=p>,</span> <span class=n>timing_state</span><span class=p>)</span> <span class=o>~</span> <span class=m>-5</span><span class=p>,</span>
                              <span class=nf>grepl</span><span class=p>(</span><span class=s>&quot;late_subclonal&quot;</span><span class=p>,</span> <span class=n>timing_state</span><span class=p>)</span> <span class=o>~</span> <span class=m>-1</span><span class=p>,</span>
                              <span class=kc>TRUE</span> <span class=o>~</span> <span class=m>0</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span> <span class=n>result</span><span class=p>,</span> <span class=n>n</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span> <span class=n>result</span><span class=p>,</span> <span class=n>n</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=c1>## get weighted counts by multiplying events and occurrence</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>weighted_result</span> <span class=o>=</span> <span class=n>result</span> <span class=o>*</span> <span class=n>n</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>rename</span><span class=p>(</span><span class=n>freq</span> <span class=o>=</span> <span class=n>n</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>group_by</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>mean_result</span> <span class=o>=</span> <span class=nf>mean</span><span class=p>(</span><span class=n>weighted_result</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>ungroup</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=nf>arrange</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span> <span class=n>mean_result</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span> <span class=n>mean_result</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=c1>## decide early vs late per driver event using</span>
    <span class=c1>## majority vote</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>timing</span> <span class=o>=</span> <span class=nf>case_when</span><span class=p>(</span><span class=n>mean_result</span> <span class=o>&lt;</span> <span class=m>0</span> <span class=o>~</span> <span class=s>&quot;LATE&quot;</span><span class=p>,</span>
                              <span class=n>mean_result</span> <span class=o>&gt;</span> <span class=m>0</span> <span class=o>~</span> <span class=s>&quot;EARLY&quot;</span><span class=p>,</span>
                              <span class=n>mean_result</span> <span class=o>==</span> <span class=m>0</span> <span class=o>~</span> <span class=s>&quot;DRAW&quot;</span><span class=p>,</span>
                              <span class=kc>TRUE</span> <span class=o>~</span> <span class=s>&quot;NONE&quot;</span><span class=p>))</span>

  <span class=c1>## let driver gene compete with one another: combination matrix</span>
  <span class=n>wintbl_combn</span> <span class=o>&lt;-</span> <span class=nf>choose</span><span class=p>(</span><span class=nf>nrow</span><span class=p>(</span><span class=n>vcf_timing</span><span class=p>),</span> <span class=m>2</span><span class=p>)</span>

  <span class=n>snv_df_wintbl</span> <span class=o>&lt;-</span> <span class=nf>as.data.frame</span><span class=p>(</span><span class=nf>matrix</span><span class=p>(</span><span class=nf>apply</span><span class=p>(</span><span class=n>vcf_timing</span> <span class=o>%&gt;%</span>
                            <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>Gene_Name</span><span class=p>,</span>
                            <span class=n>mean_result</span><span class=p>),</span>
                        <span class=m>2</span><span class=p>,</span> <span class=n>combn</span><span class=p>,</span> <span class=n>m</span> <span class=o>=</span> <span class=m>2</span><span class=p>),</span>
                    <span class=n>nrow</span> <span class=o>=</span> <span class=n>wintbl_combn</span><span class=p>),</span>
                    <span class=n>stringsAsFactors</span> <span class=o>=</span> <span class=bp>F</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>tbl_df</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=c1>## rename V1 and V2 to P1 and P2, respectively</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>rename_at</span><span class=p>(</span><span class=nf>vars</span><span class=p>(</span><span class=n>V1</span><span class=p>,</span> <span class=n>V2</span><span class=p>),</span> <span class=o>~</span><span class=nf>gsub</span><span class=p>(</span><span class=s>&quot;V&quot;</span><span class=p>,</span> <span class=s>&quot;P&quot;</span><span class=p>,</span> <span class=n>.</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=c1>## where _vaf is clonality based estimate if event</span>
    <span class=c1>## is ~clonal (early) or ~subclonal</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>rename</span><span class=p>(</span><span class=n>P1_vaf</span> <span class=o>=</span> <span class=n>V3</span><span class=p>,</span>
           <span class=n>P2_vaf</span> <span class=o>=</span> <span class=n>V4</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=c1>## take a win/loss/draw call</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>outcome</span> <span class=o>=</span> <span class=nf>case_when</span><span class=p>(</span><span class=n>P1_vaf</span> <span class=o>&gt;</span> <span class=n>P2_vaf</span> <span class=o>~</span> <span class=s>&quot;P1&quot;</span><span class=p>,</span>
                               <span class=n>P1_vaf</span> <span class=o>&lt;</span> <span class=n>P2_vaf</span> <span class=o>~</span> <span class=s>&quot;P2&quot;</span><span class=p>,</span>
                               <span class=n>P1_vaf</span> <span class=o>==</span> <span class=n>P2_vaf</span> <span class=o>~</span> <span class=s>&quot;D&quot;</span><span class=p>,</span>
                               <span class=kc>TRUE</span> <span class=o>~</span> <span class=kc>NA_character_</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>P1</span><span class=p>,</span> <span class=n>P2</span><span class=p>,</span> <span class=n>outcome</span><span class=p>)</span>

  <span class=nf>return</span><span class=p>(</span><span class=n>snv_df_wintbl</span><span class=p>)</span>
<span class=p>}</span>

<span class=c1>## Get winning tables from vcfs</span>
<span class=n>cg_win_table</span> <span class=o>&lt;-</span> <span class=nf>mk_win_table</span><span class=p>(</span><span class=n>win_table_cg_vcf</span><span class=p>)</span>
<span class=n>pg_win_table</span> <span class=o>&lt;-</span> <span class=nf>mk_win_table</span><span class=p>(</span><span class=n>win_table_pg_vcf</span><span class=p>)</span>
<span class=n>ag_win_table</span> <span class=o>&lt;-</span> <span class=nf>mk_win_table</span><span class=p>(</span><span class=n>win_table_ag_vcf</span><span class=p>)</span>
</code></pre></div> <h3 id=fig-3d>Fig 3D<a class=headerlink href=#fig-3d title="Permanent link">&para;</a></h3> <blockquote> <p>Run <em>Bradley-Terry model</em> to get <a href=/methods/S14_mut_timing/ >estimated relative timing of somatic driver events</a>.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>#### Run and plot BT model ####</span>
<span class=c1>## default to Canine Glioma</span>
<span class=n>plot_timing_btmodel</span> <span class=o>&lt;-</span> <span class=nf>function</span><span class=p>(</span><span class=n>win_table</span> <span class=o>=</span> <span class=n>cg_win_table</span><span class=p>,</span>
                                <span class=n>plot_title</span> <span class=o>=</span> <span class=s>&quot;Canine Glioma&quot;</span><span class=p>,</span>
                                <span class=n>genes2plot</span> <span class=o>=</span> <span class=n>oncoprint_genes_snv_scna</span><span class=p>)</span> <span class=p>{</span>

  <span class=nf>library</span><span class=p>(</span><span class=n>tidyverse</span><span class=p>)</span>
  <span class=nf>library</span><span class=p>(</span><span class=n>BradleyTerryScalable</span><span class=p>)</span>

  <span class=nf>library</span><span class=p>(</span><span class=n>igraph</span><span class=p>)</span>
  <span class=nf>library</span><span class=p>(</span><span class=n>ggridges</span><span class=p>)</span>

  <span class=nf>set.seed</span><span class=p>(</span><span class=m>1989</span><span class=p>)</span>

  <span class=c1>#### Import somatic winning table ####</span>
  <span class=n>win_table_raw</span> <span class=o>&lt;-</span> <span class=n>win_table</span>

  <span class=c1>#### create btdata ####       </span>
  <span class=n>toy_data_4col</span> <span class=o>&lt;-</span> <span class=nf>codes_to_counts</span><span class=p>(</span><span class=n>win_table_raw</span><span class=p>,</span> <span class=nf>c</span><span class=p>(</span><span class=s>&quot;P1&quot;</span><span class=p>,</span> <span class=s>&quot;P2&quot;</span><span class=p>,</span> <span class=s>&quot;D&quot;</span><span class=p>))</span>
  <span class=n>toy_btdata</span> <span class=o>&lt;-</span> <span class=nf>btdata</span><span class=p>(</span><span class=n>toy_data_4col</span><span class=p>,</span> <span class=n>return_graph</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

  <span class=c1>#### fit model: btfit #### </span>
  <span class=c1>## Prefer using MAP estimate if summary(toy_btdata) returns</span>
  <span class=c1>## not-fully connected network: Needs to be checked manually</span>
  <span class=c1>## and to follow warning flags</span>
  <span class=nf>print</span><span class=p>(</span><span class=nf>summary</span><span class=p>(</span><span class=n>toy_btdata</span><span class=p>))</span>

  <span class=n>toy_fit_MLE</span> <span class=o>&lt;-</span> <span class=nf>btfit</span><span class=p>(</span><span class=n>toy_btdata</span><span class=p>,</span> <span class=m>1</span><span class=p>)</span>
  <span class=n>toy_fit_MAP</span> <span class=o>&lt;-</span> <span class=nf>btfit</span><span class=p>(</span><span class=n>toy_btdata</span><span class=p>,</span> <span class=m>1.1</span><span class=p>)</span>

  <span class=nf>summary</span><span class=p>(</span><span class=n>toy_fit_MLE</span><span class=p>,</span> <span class=n>SE</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>
  <span class=nf>summary</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>,</span> <span class=n>SE</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

  <span class=nf>coef</span><span class=p>(</span><span class=n>toy_fit_MLE</span><span class=p>)</span>
  <span class=nf>coef</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>)</span>

  <span class=nf>vcov</span><span class=p>(</span><span class=n>toy_fit_MLE</span><span class=p>,</span> <span class=n>ref</span> <span class=o>=</span> <span class=s>&quot;home&quot;</span><span class=p>)</span>
  <span class=nf>vcov</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>,</span> <span class=n>ref</span> <span class=o>=</span> <span class=s>&quot;home&quot;</span><span class=p>)</span>

  <span class=c1>#### get winning prob ####</span>

  <span class=nf>fitted</span><span class=p>(</span><span class=n>toy_fit_MLE</span><span class=p>,</span> <span class=n>as_df</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>
  <span class=nf>fitted</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>,</span> <span class=n>as_df</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

  <span class=nf>btprob</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>,</span> <span class=n>as_df</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>
  <span class=nf>btprob</span><span class=p>(</span><span class=n>toy_fit_MLE</span><span class=p>,</span> <span class=n>as_df</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

  <span class=n>toy_fit_MAP_prob</span> <span class=o>&lt;-</span> <span class=nf>btprob</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>)</span>
  <span class=n>toy_fit_map_df</span> <span class=o>&lt;-</span> <span class=nf>btprob</span><span class=p>(</span><span class=n>toy_fit_MAP</span><span class=p>,</span> <span class=n>as_df</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

  <span class=n>toy_fit_map_mat</span> <span class=o>&lt;-</span> <span class=nf>as.data.frame</span><span class=p>(</span><span class=nf>as.matrix</span><span class=p>(</span><span class=n>toy_fit_MAP_prob</span><span class=p>))</span>

  <span class=c1>## convert to rank-order df</span>
  <span class=n>toy_fit_map_mat_ranked</span> <span class=o>&lt;-</span> <span class=n>toy_fit_map_mat</span> <span class=o>%&gt;%</span>
    <span class=nf>rownames_to_column</span><span class=p>(</span><span class=s>&quot;driver&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate_at</span><span class=p>(</span><span class=nf>vars</span><span class=p>(</span><span class=o>-</span><span class=n>driver</span><span class=p>),</span> <span class=o>~</span><span class=p>(</span><span class=m>1</span> <span class=o>-</span> <span class=nf>cume_dist</span><span class=p>(</span><span class=n>.</span><span class=p>)))</span>

  <span class=c1>## probability long table</span>
  <span class=n>toy_fit_map_mat_long_tbl</span> <span class=o>&lt;-</span> <span class=n>toy_fit_map_mat</span> <span class=o>%&gt;%</span>
    <span class=nf>rownames_to_column</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>p2</span><span class=p>,</span> <span class=n>prob</span><span class=p>,</span> <span class=o>-</span><span class=n>rowname</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>()</span>

  <span class=c1>## rank order long table</span>
  <span class=n>toy_fit_map_mat_ranked_long_tbl</span> <span class=o>&lt;-</span> <span class=n>toy_fit_map_mat_ranked</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>p2</span><span class=p>,</span> <span class=n>rank</span><span class=p>,</span> <span class=o>-</span><span class=n>driver</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>()</span>

  <span class=c1>#### Figure 3D: Timing Plots ####</span>

  <span class=n>toy_fit_map_mat_long_tbl</span> <span class=o>&lt;-</span> <span class=n>toy_fit_map_mat</span> <span class=o>%&gt;%</span>
    <span class=nf>rownames_to_column</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=c1>## select rows to plot matching genes found in</span>
    <span class=c1>## known somatic driver genes</span>
    <span class=nf>filter</span><span class=p>(</span><span class=n>rowname</span> <span class=o>%in%</span> <span class=n>genes2plot</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>p2</span><span class=p>,</span> <span class=n>prob</span><span class=p>,</span> <span class=o>-</span><span class=n>rowname</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>()</span>

  <span class=c1>#### export timing plots ####</span>
  <span class=nf>ggplot</span><span class=p>(</span><span class=n>toy_fit_map_mat_long_tbl</span><span class=p>,</span>
    <span class=nf>aes</span><span class=p>(</span><span class=m>1</span><span class=o>-</span><span class=n>prob</span><span class=p>,</span> <span class=n>rowname</span><span class=p>,</span> <span class=n>fill</span> <span class=o>=</span> <span class=n>..x..</span><span class=p>),</span> <span class=n>group</span> <span class=o>=</span> <span class=n>p2</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>geom_density_ridges_gradient</span><span class=p>(</span><span class=n>scale</span> <span class=o>=</span> <span class=m>1</span><span class=p>,</span> <span class=n>rel_min_height</span> <span class=o>=</span> <span class=m>0.01</span><span class=p>)</span> <span class=o>+</span>
    <span class=n>viridis</span><span class=o>::</span><span class=nf>scale_fill_viridis</span><span class=p>(</span><span class=n>name</span> <span class=o>=</span> <span class=s>&quot;Probability&quot;</span><span class=p>,</span> <span class=n>option</span> <span class=o>=</span> <span class=s>&quot;C&quot;</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>labs</span><span class=p>(</span><span class=n>title</span> <span class=o>=</span> <span class=n>plot_title</span><span class=p>,</span>
         <span class=n>x</span> <span class=o>=</span> <span class=s>&quot;Probability being a late event&quot;</span><span class=p>,</span>
         <span class=n>y</span> <span class=o>=</span> <span class=s>&quot;Somatic Drivers&quot;</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>guides</span><span class=p>(</span><span class=n>fill</span> <span class=o>=</span> <span class=nf>guide_colorbar</span><span class=p>(</span><span class=n>barwidth</span> <span class=o>=</span> <span class=m>15</span><span class=p>,</span>
                                 <span class=n>barheight</span> <span class=o>=</span> <span class=m>1.5</span><span class=p>))</span> <span class=o>+</span>
    <span class=nf>theme_ridges</span><span class=p>(</span><span class=n>font_size</span> <span class=o>=</span> <span class=m>22</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>legend.position</span><span class=o>=</span><span class=s>&quot;bottom&quot;</span><span class=p>)</span>

<span class=p>}</span>

<span class=c1>## plot per-cohort relative timing probabilities of driver genes</span>
<span class=n>timing_plot_canine</span> <span class=o>&lt;-</span> <span class=nf>plot_timing_btmodel</span><span class=p>(</span><span class=n>win_table</span> <span class=o>=</span> <span class=n>cg_win_table</span><span class=p>,</span>
                                <span class=n>plot_title</span> <span class=o>=</span> <span class=s>&quot;Canine Glioma&quot;</span><span class=p>,</span>
                                <span class=n>genes2plot</span> <span class=o>=</span> <span class=n>oncoprint_genes_snv_scna</span><span class=p>)</span>

<span class=n>timing_plot_peds</span> <span class=o>&lt;-</span> <span class=nf>plot_timing_btmodel</span><span class=p>(</span><span class=n>win_table</span> <span class=o>=</span> <span class=n>pg_win_table</span><span class=p>,</span>
                                <span class=n>plot_title</span> <span class=o>=</span> <span class=s>&quot;Human Pediatric Glioma&quot;</span><span class=p>,</span>
                                <span class=n>genes2plot</span> <span class=o>=</span> <span class=n>oncoprint_genes_snv_scna</span><span class=p>)</span>

<span class=n>timing_plot_adult</span> <span class=o>&lt;-</span> <span class=nf>plot_timing_btmodel</span><span class=p>(</span><span class=n>win_table</span> <span class=o>=</span> <span class=n>ag_win_table</span><span class=p>,</span>
                                <span class=n>plot_title</span> <span class=o>=</span> <span class=s>&quot;human Adult Glioma&quot;</span><span class=p>,</span>
                                <span class=n>genes2plot</span> <span class=o>=</span> <span class=n>oncoprint_genes_snv_scna</span><span class=p>)</span>

<span class=nf>library</span><span class=p>(</span><span class=n>gridExtra</span><span class=p>)</span>

<span class=nf>ggsave</span><span class=p>(</span><span class=s>&quot;F3D.pdf&quot;</span><span class=p>,</span>
    <span class=nf>grid.arrange</span><span class=p>(</span><span class=n>timing_plot_canine</span><span class=p>,</span>
                <span class=n>timing_plot_peds</span><span class=p>,</span>
                <span class=n>timing_plot_adult</span><span class=p>,</span>
                <span class=n>ncol</span> <span class=o>=</span> <span class=m>3</span><span class=p>,</span>
                <span class=n>widths</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(</span><span class=m>4</span><span class=p>,</span><span class=m>4</span><span class=p>,</span><span class=m>4</span><span class=p>),</span> <span class=n>heights</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(</span><span class=m>4</span><span class=p>,</span><span class=m>4</span><span class=p>,</span><span class=m>4</span><span class=p>)),</span>
    <span class=n>width</span> <span class=o>=</span> <span class=m>24</span><span class=p>,</span>
    <span class=n>height</span> <span class=o>=</span> <span class=m>24</span><span class=p>,</span>
    <span class=n>useDingbats</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>
</code></pre></div> <div class=footnote> <hr> <ol> <li id=fn:1> <p>Shinde J, Bayard Q, Imbeaud S, Hirsch TZ, Liu F, Renault V, et al. Palimpsest: an R package for studying mutational and structural variant signatures along clonal evolution in cancer. Bioinformatics 2018. doi: <a href=https://doi.org/10.1093/bioinformatics/bty388>10.1093/bioinformatics/bty388</a>.&#160;<a class=footnote-backref href=#fnref:1 title="Jump back to footnote 1 in the text">&#8617;</a></p> </li> </ol> </div> <hr> <div class=md-source-date> <small> Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">February 7, 2020</span> </small> </div> <!--   
    <p><small>Author: Samir B. Amin</small></p>
   --> <h2 id=__comments>Comments</h2> <div id=disqus_thread></div> <script>var disqus_config=function(){this.page.url="https://canineglioma.verhaaklab.com/figures/F3D/",this.page.identifier="figures/F3D/"};window.addEventListener("load",function(){var e=document,i=e.createElement("script");i.src="//verhaaklab.disqus.com/embed.js",i.setAttribute("data-timestamp",+new Date),(e.head||e.body).appendChild(i)})</script> </article> </div> </div> </main> <footer class=md-footer> <nav class="md-footer__inner md-grid" aria-label=Footer> <a href=../F3C/ class="md-footer__link md-footer__link--prev" aria-label="Previous: F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes" rel=prev> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </div> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Previous </span> F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes </div> </div> </a> <a href=../S3/ class="md-footer__link md-footer__link--next" aria-label="Next: S3 &middot; Suppl Figures" rel=next> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Next </span> S3 &middot; Suppl Figures </div> </div> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg> </div> </a> </nav> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <div class=md-footer-copyright> <div class=md-footer-copyright__highlight> Copyright &copy; 2017 - 2021 <a href=https://verhaaklab.com>Verhaak Lab at the Jackson Laboratory for Genomic Medicine</a> </div> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> <div class=md-footer-social> <a href=https://verhaaklab.com/contact target=_blank rel=noopener title=verhaaklab.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M464 64H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V112c0-26.51-21.49-48-48-48zm0 48v40.805c-22.422 18.259-58.168 46.651-134.587 106.49-16.841 13.247-50.201 45.072-73.413 44.701-23.208.375-56.579-31.459-73.413-44.701C106.18 199.465 70.425 171.067 48 152.805V112h416zM48 400V214.398c22.914 18.251 55.409 43.862 104.938 82.646 21.857 17.205 60.134 55.186 103.062 54.955 42.717.231 80.509-37.199 103.053-54.947 49.528-38.783 82.032-64.401 104.947-82.653V400H48z"/></svg> </a> <a href=https://twitter.com/roelverhaak target=_blank rel=noopener title=twitter.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> <a href=https://github.com/sbamin target=_blank rel=noopener title=github.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg> </a> <a href=https://twitter.com/sbamin target=_blank rel=noopener title=twitter.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "../..", "features": ["tabs", "instant", "search.suggest", "search.highlight", "search.share"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../assets/javascripts/workers/search.fcfe8b6d.min.js", "version": null}</script> <script src=../../assets/javascripts/bundle.b1047164.min.js></script> <script src=/assets/js/extra.js></script> <script src=/assets/js/katex.js></script> <script src=https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.js></script> <script src=https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/contrib/auto-render.min.js></script> <script src=https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js></script> <script src=/assets/js/tables.js></script> </body> </html>