<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=author content="Samir B. Amin"><link href=https://canineglioma.verhaaklab.com/figures/F3A/ rel=canonical><link rel=icon href=../../assets/images/fixed/favicon.png><meta name=generator content="mkdocs-1.2.3, mkdocs-material-8.2.5"><title>F3A &middot; Somatic Mutational Signatures - Deconvoluted - Canine Glioma Project &middot; Verhaak Lab</title><link rel=stylesheet href=../../assets/stylesheets/main.2d9f7617.min.css><link rel=stylesheet href=../../assets/stylesheets/palette.e6a45f82.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CFira+Code:400,400i,700,700i&display=fallback"><style>:root{--md-text-font:"Roboto";--md-code-font:"Fira Code"}</style><link rel=stylesheet href=../../assets/css/extra.css><script>__md_scope=new URL("../..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script><!-- Determine analytics provider (deprecated, removed in v8) --><!-- Determine analytics provider --><!-- Set up analytics provider --><!-- Determine analytics property (deprecated, removed in v8) --><!-- Determine analytics property --><!-- Google Analytics 4 (G-XXXXXXXXXX) --><script>
    window.ga = window.ga || function() {
      (ga.q = ga.q || []).push(arguments)
    }
    ga.l = +new Date()

    /* Set up integration and send page view */
    ga("create", "UA-64053990-2", "auto")
    ga("set", "anonymizeIp", true)
    ga("send", "pageview")

    /* Register virtual event handlers */
    document.addEventListener("DOMContentLoaded", function() {
      if (document.forms.search) {
        var query = document.forms.search.query
        query.addEventListener("blur", function() {
          if (this.value) {
            var path = document.location.pathname;
            ga("send", "pageview", path + "?q=" + this.value)
          }
        })
      }

      /* Send page view on location change */
      if (typeof location$ !== "undefined")
        location$.subscribe(function(url) {
          ga("send", "pageview", url.pathname)
        })
    })
  </script><script async src=https://www.google-analytics.com/analytics.js></script><!--   <link rel="stylesheet" href="https://canineglioma.verhaaklab.com/../../assets/stylesheets/main.c296cc61.min.css"> --><meta name=keywords content="canine,glioma,pediatric,comparative genomics,DIPG,verhaak,lab,jax"><meta property=og:type content=website><meta property=og:title content="F3A · Somatic Mutational Signatures - Deconvoluted · Canine Glioma Project · Verhaak Lab"><meta property=og:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta content=https://canineglioma.verhaaklab.com/figures/F3A/ property=og:url><meta property=og:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta property=og:image:type content=image/png><meta property=og:image:width content=256><meta property=og:image:height content=256><meta name=twitter:card content=summary><meta name=twitter:site content=@sbamin><meta name=twitter:creator content=@sbamin><meta name=twitter:title content="F3A · Somatic Mutational Signatures - Deconvoluted · Canine Glioma Project · Verhaak Lab"><meta name=twitter:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=twitter:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta name=twitter:image:alt content="webpage banner or site logo"></head> <body dir=ltr data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo> <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#required-metadata class=md-skip> Skip to content </a> </div> <div data-md-component=announce> </div> <div data-md-component=outdated hidden> <aside class="md-banner md-banner--warning"> </aside> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-header__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> Canine Glioma Project &middot; Verhaak Lab </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> F3A &middot; Somatic Mutational Signatures - Deconvoluted </span> </div> </div> </div> <form class=md-header__option data-md-component=palette> <input class=md-option data-md-color-media data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo aria-label="Switch to dark mode" type=radio name=__palette id=__palette_1> <label class="md-header__button md-icon" title="Switch to dark mode" for=__palette_2 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg> </label> <input class=md-option data-md-color-media data-md-color-scheme=slate data-md-color-primary=teal data-md-color-accent=light-green aria-label="Switch to light mode" type=radio name=__palette id=__palette_2> <label class="md-header__button md-icon" title="Switch to light mode" for=__palette_1 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3z"/></svg> </label> </form> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </label> <nav class=md-search__options aria-label=Search> <button type=reset class="md-search__icon md-icon" aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg> </button> </nav> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M100.59 334.24c48.57 3.31 58.95 2.11 58.95 11.94 0 20-65.55 20.06-65.55 1.52.01-5.09 3.29-9.4 6.6-13.46zm27.95-116.64c-32.29 0-33.75 44.47-.75 44.47 32.51 0 31.71-44.47.75-44.47zM448 80v352a48 48 0 0 1-48 48H48a48 48 0 0 1-48-48V80a48 48 0 0 1 48-48h352a48 48 0 0 1 48 48zm-227 69.31c0 14.49 8.38 22.88 22.86 22.88 14.74 0 23.13-8.39 23.13-22.88S258.62 127 243.88 127c-14.48 0-22.88 7.84-22.88 22.31zM199.18 195h-49.55c-25-6.55-81.56-4.85-81.56 46.75 0 18.8 9.4 32 21.85 38.11C74.23 294.23 66.8 301 66.8 310.6c0 6.87 2.79 13.22 11.18 16.76-8.9 8.4-14 14.48-14 25.92C64 373.35 81.53 385 127.52 385c44.22 0 69.87-16.51 69.87-45.73 0-36.67-28.23-35.32-94.77-39.38l8.38-13.43c17 4.74 74.19 6.23 74.19-42.43 0-11.69-4.83-19.82-9.4-25.67l23.38-1.78zm84.34 109.84-13-1.78c-3.82-.51-4.07-1-4.07-5.09V192.52h-52.6l-2.79 20.57c15.75 5.55 17 4.86 17 10.17V298c0 5.62-.31 4.58-17 6.87v20.06h72.42zM384 315l-6.87-22.37c-40.93 15.37-37.85-12.41-37.85-16.73v-60.72h37.85v-25.41h-35.82c-2.87 0-2 2.52-2-38.63h-24.18c-2.79 27.7-11.68 38.88-34 41.42v22.62c20.47 0 19.82-.85 19.82 2.54v66.57c0 28.72 11.43 40.91 41.67 40.91 14.45 0 30.45-4.83 41.38-10.2z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class="md-tabs__inner md-grid"> <ul class=md-tabs__list> <li class=md-tabs__item> <a href=../.. class=md-tabs__link> Home </a> </li> <li class=md-tabs__item> <a href=../../methods/S01_samples/ class=md-tabs__link> Methods </a> </li> <li class=md-tabs__item> <a href=../A1_preload/ class="md-tabs__link md-tabs__link--active"> Figures </a> </li> <li class=md-tabs__item> <a href=../../run_code/ class=md-tabs__link> R Markdown </a> </li> <li class=md-tabs__item> <a href=https://verhaaklab.com/contact class=md-tabs__link> Contact </a> </li> </ul> </div> </nav> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-nav__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> Canine Glioma Project &middot; Verhaak Lab </label> <div class=md-nav__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M100.59 334.24c48.57 3.31 58.95 2.11 58.95 11.94 0 20-65.55 20.06-65.55 1.52.01-5.09 3.29-9.4 6.6-13.46zm27.95-116.64c-32.29 0-33.75 44.47-.75 44.47 32.51 0 31.71-44.47.75-44.47zM448 80v352a48 48 0 0 1-48 48H48a48 48 0 0 1-48-48V80a48 48 0 0 1 48-48h352a48 48 0 0 1 48 48zm-227 69.31c0 14.49 8.38 22.88 22.86 22.88 14.74 0 23.13-8.39 23.13-22.88S258.62 127 243.88 127c-14.48 0-22.88 7.84-22.88 22.31zM199.18 195h-49.55c-25-6.55-81.56-4.85-81.56 46.75 0 18.8 9.4 32 21.85 38.11C74.23 294.23 66.8 301 66.8 310.6c0 6.87 2.79 13.22 11.18 16.76-8.9 8.4-14 14.48-14 25.92C64 373.35 81.53 385 127.52 385c44.22 0 69.87-16.51 69.87-45.73 0-36.67-28.23-35.32-94.77-39.38l8.38-13.43c17 4.74 74.19 6.23 74.19-42.43 0-11.69-4.83-19.82-9.4-25.67l23.38-1.78zm84.34 109.84-13-1.78c-3.82-.51-4.07-1-4.07-5.09V192.52h-52.6l-2.79 20.57c15.75 5.55 17 4.86 17 10.17V298c0 5.62-.31 4.58-17 6.87v20.06h72.42zM384 315l-6.87-22.37c-40.93 15.37-37.85-12.41-37.85-16.73v-60.72h37.85v-25.41h-35.82c-2.87 0-2 2.52-2-38.63h-24.18c-2.79 27.7-11.68 38.88-34 41.42v22.62c20.47 0 19.82-.85 19.82 2.54v66.57c0 28.72 11.43 40.91 41.67 40.91 14.45 0 30.45-4.83 41.38-10.2z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../.. class=md-nav__link> Home </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_2 type=checkbox id=__nav_2> <label class=md-nav__link for=__nav_2> Methods <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Methods data-md-level=1> <label class=md-nav__title for=__nav_2> <span class="md-nav__icon md-icon"></span> Methods </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../methods/S01_samples/ class=md-nav__link> Samples </a> </li> <li class=md-nav__item> <a href=../../methods/S02_data_sources/ class=md-nav__link> Data Sources </a> </li> <li class=md-nav__item> <a href=../../methods/S03_sample_prep/ class=md-nav__link> Sample Preparations </a> </li> <li class=md-nav__item> <a href=../../methods/S04_alignments_fprint/ class=md-nav__link> Alignments </a> </li> <li class=md-nav__item> <a href=../../methods/S05_somatic_variants/ class=md-nav__link> Somatic SNVs and Indels </a> </li> <li class=md-nav__item> <a href=../../methods/S06_smgs/ class=md-nav__link> Significantly Mutated Genes </a> </li> <li class=md-nav__item> <a href=../../methods/S07_hallmarks/ class=md-nav__link> Cancer Hallmarks </a> </li> <li class=md-nav__item> <a href=../../methods/S09_mutation_rate/ class=md-nav__link> Somatic Mutation Rate </a> </li> <li class=md-nav__item> <a href=../../methods/S10_SCNA/ class=md-nav__link> Somatic Copy Number Aberrations </a> </li> <li class=md-nav__item> <a href=../../methods/S11_aneuploidy/ class=md-nav__link> Aneuploidy Metrics </a> </li> <li class=md-nav__item> <a href=../../methods/S12_ith/ class=md-nav__link> Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../../methods/S13_mut_sigs/ class=md-nav__link> Mutational Signatures </a> </li> <li class=md-nav__item> <a href=../../methods/S14_mut_timing/ class=md-nav__link> Molecular Timing of Somatic Drivers </a> </li> <li class=md-nav__item> <a href=../../methods/S15_class_predict_methy/ class=md-nav__link> Class Prediction using methylation classifier </a> </li> <li class=md-nav__item> <a href=../../methods/S16_expression_immune/ class=md-nav__link> Immune Microenvironment Analysis </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3 type=checkbox id=__nav_3 checked> <label class=md-nav__link for=__nav_3> Figures <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Figures data-md-level=1> <label class=md-nav__title for=__nav_3> <span class="md-nav__icon md-icon"></span> Figures </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../A1_preload/ class=md-nav__link> Start Here </a> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_2 type=checkbox id=__nav_3_2> <label class=md-nav__link for=__nav_3_2> F1 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F1 data-md-level=2> <label class=md-nav__title for=__nav_3_2> <span class="md-nav__icon md-icon"></span> F1 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F1A/ class=md-nav__link> F1A &middot; Somatic Mutational Landscape </a> </li> <li class=md-nav__item> <a href=../F1B/ class=md-nav__link> F1B &middot; Somatic Recurrent Hotspot Mutations </a> </li> <li class=md-nav__item> <a href=../F1C/ class=md-nav__link> F1C &middot; Comparative Pathway Analysis </a> </li> <li class=md-nav__item> <a href=../F1D/ class=md-nav__link> F1D &middot; Comparative Somatic Mutational Burden </a> </li> <li class=md-nav__item> <a href=../S1/ class=md-nav__link> S1A &middot; DNA fingerprinting </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_3 type=checkbox id=__nav_3_3> <label class=md-nav__link for=__nav_3_3> F2 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F2 data-md-level=2> <label class=md-nav__title for=__nav_3_3> <span class="md-nav__icon md-icon"></span> F2 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F2A/ class=md-nav__link> F2A &middot; Somatic Copy Number Landscape </a> </li> <li class=md-nav__item> <a href=../F2B/ class=md-nav__link> F2B &middot; Comparative Aneuploidy Burden </a> </li> <li class=md-nav__item> <a href=../F2C/ class=md-nav__link> F2C &middot; Comparative Shared Syntenic Aneuploidy </a> </li> <li class=md-nav__item> <a href=../F2D/ class=md-nav__link> F2D &middot; Comparative Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../S2/ class=md-nav__link> S2 &middot; IGV Plots </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_4 type=checkbox id=__nav_3_4 checked> <label class=md-nav__link for=__nav_3_4> F3 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F3 data-md-level=2> <label class=md-nav__title for=__nav_3_4> <span class="md-nav__icon md-icon"></span> F3 </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" data-md-toggle=toc type=checkbox id=__toc> <label class="md-nav__link md-nav__link--active" for=__toc> F3A &middot; Somatic Mutational Signatures - Deconvoluted <span class="md-nav__icon md-icon"></span> </label> <a href=./ class="md-nav__link md-nav__link--active"> F3A &middot; Somatic Mutational Signatures - Deconvoluted </a> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#required-metadata class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-3a class=md-nav__link> Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3a class=md-nav__link> Suppl Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3b class=md-nav__link> Suppl Fig 3B </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=../F3B/ class=md-nav__link> F3B &middot; Somatic Mutational Landscape - de novo </a> </li> <li class=md-nav__item> <a href=../F3C/ class=md-nav__link> F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes </a> </li> <li class=md-nav__item> <a href=../F3D/ class=md-nav__link> F3D &middot; Comparative Molecular Timing of Driver Events </a> </li> <li class=md-nav__item> <a href=../S3/ class=md-nav__link> S3 &middot; Suppl Figures </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_5 type=checkbox id=__nav_3_5> <label class=md-nav__link for=__nav_3_5> F4 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F4 data-md-level=2> <label class=md-nav__title for=__nav_3_5> <span class="md-nav__icon md-icon"></span> F4 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F4/ class=md-nav__link> F4 &middot; DNA Methylation based Prediction of Canine Gliomas </a> </li> <li class=md-nav__item> <a href=../S4/ class=md-nav__link> S4 &middot; Suppl Figure 4 </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_6 type=checkbox id=__nav_3_6> <label class=md-nav__link for=__nav_3_6> F5 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F5 data-md-level=2> <label class=md-nav__title for=__nav_3_6> <span class="md-nav__icon md-icon"></span> F5 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../F5/ class=md-nav__link> F5B &middot; Comparative Immunohistochemistry - Quantitative </a> </li> <li class=md-nav__item> <a href=../S5/ class=md-nav__link> S5 &middot; Suppl Figure 5 </a> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_4 type=checkbox id=__nav_4> <div class="md-nav__link md-nav__link--index "> <a href=../../run_code/ >R Markdown</a> <label for=__nav_4> <span class="md-nav__icon md-icon"></span> </label> </div> <nav class=md-nav aria-label="R Markdown" data-md-level=1> <label class=md-nav__title for=__nav_4> <span class="md-nav__icon md-icon"></span> R Markdown </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../run_code/run_all/ class=md-nav__link> Output </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=https://verhaaklab.com/contact class=md-nav__link> Contact </a> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=sidebar data-md-type=toc> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#required-metadata class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-3a class=md-nav__link> Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3a class=md-nav__link> Suppl Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3b class=md-nav__link> Suppl Fig 3B </a> </li> </ul> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <!-- Add copied contents from original content.html here --> <!-- https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/src/partials/content.html --> <!-- Edit button --> <!--
  Hack: check whether the content contains a h1 headline. If it
  doesn't, the page title (or respectively site name) is used
  as the main headline.
--> <h1>F3A &middot; Somatic Mutational Signatures - Deconvoluted</h1> <!-- Markdown content --> <h3 id=required-metadata>Required metadata<a class=headerlink href=#required-metadata title="Permanent link">&para;</a></h3> <ul> <li>Canine glioma patients with <a href=/methods/S13_mut_sigs/ >outlier mutation profile</a>.</li> </ul> <div class=highlight><pre><span></span><code><span class=c1>## filter out missing values in mutation rate column</span>
<span class=n>cgp_sample_info_maftools_hypermuts</span> <span class=o>&lt;-</span> <span class=n>cgp_suppl_sample_info_master</span> <span class=o>%&gt;%</span>
  <span class=nf>filter</span><span class=p>(</span><span class=o>!</span><span class=nf>is.na</span><span class=p>(</span><span class=n>adusted_coding_mutrate</span><span class=p>))</span> <span class=o>%&gt;%</span>
  <span class=nf>arrange</span><span class=p>(</span><span class=nf>desc</span><span class=p>(</span><span class=n>adusted_coding_mutrate</span><span class=p>))</span>

<span class=n>outlier_model_hypermut</span> <span class=o>&lt;-</span> <span class=nf>lm</span><span class=p>(</span><span class=n>adusted_coding_mutrate</span> <span class=o>~</span> <span class=n>Matched_Normal</span> <span class=o>+</span>
    <span class=n>Tissue_Archival</span><span class=p>,</span> <span class=n>data</span> <span class=o>=</span> <span class=n>cgp_sample_info_maftools_hypermuts</span><span class=p>)</span>

<span class=nf>summary</span><span class=p>(</span><span class=n>outlier_model_hypermut</span><span class=p>)</span>

<span class=c1>## Hypermut outliers after correcting for archival and paired vs tonly mode</span>
<span class=n>hypermut_true_outliers</span> <span class=o>&lt;-</span> <span class=n>car</span><span class=o>::</span><span class=nf>outlierTest</span><span class=p>(</span><span class=n>outlier_model_hypermut</span><span class=p>,</span>
    <span class=n>cutoff</span> <span class=o>=</span> <span class=m>0.05</span><span class=p>,</span>
    <span class=n>labels</span> <span class=o>=</span> <span class=n>cgp_sample_info_maftools_hypermuts</span><span class=o>$</span><span class=n>Tumor_Sample_Barcode</span><span class=p>)</span>

<span class=nf>sprintf</span><span class=p>(</span><span class=s>&quot;hypermut case: %s&quot;</span><span class=p>,</span> <span class=nf>names</span><span class=p>(</span><span class=n>hypermut_true_outliers</span><span class=o>$</span><span class=n>rstudent</span><span class=p>))</span>

<span class=n>cgp_sample_info_non_hypermut</span> <span class=o>&lt;-</span> <span class=n>cgp_sample_info_maftools_hypermuts</span> <span class=o>%&gt;%</span> 
    <span class=nf>filter</span><span class=p>(</span><span class=o>!</span><span class=n>Tumor_Sample_Barcode</span> <span class=o>%in%</span>
    <span class=nf>names</span><span class=p>(</span><span class=n>hypermut_true_outliers</span><span class=o>$</span><span class=n>rstudent</span><span class=p>))</span>

<span class=c1>#### outliers in non-hypermut cases</span>
<span class=c1>## cases with mut rate exceeding chi sq distribution by &gt; 95% </span>
<span class=n>outlier_in_non_hypermut_chisq_95</span> <span class=o>&lt;-</span> <span class=n>cgp_sample_info_non_hypermut</span><span class=p>[</span>
    <span class=n>outliers</span><span class=o>::</span><span class=nf>scores</span><span class=p>(</span><span class=n>cgp_sample_info_non_hypermut</span><span class=o>$</span><span class=n>adusted_coding_mutrate</span><span class=p>,</span>
    <span class=n>type</span> <span class=o>=</span> <span class=s>&quot;chisq&quot;</span><span class=p>,</span> <span class=n>prob</span> <span class=o>=</span> <span class=m>0.95</span><span class=p>),]</span> <span class=o>%&gt;%</span>
    <span class=nf>pull</span><span class=p>(</span><span class=n>Tumor_Sample_Barcode</span><span class=p>)</span>

<span class=nf>sprintf</span><span class=p>(</span><span class=s>&quot;outlier among non_hypermut: %s&quot;</span><span class=p>,</span> <span class=n>outlier_in_non_hypermut_chisq_95</span><span class=p>)</span>

<span class=n>cgp_outlier_mutant_cases</span> <span class=o>&lt;-</span> <span class=nf>c</span><span class=p>(</span><span class=nf>names</span><span class=p>(</span><span class=n>hypermut_true_outliers</span><span class=o>$</span><span class=n>rstudent</span><span class=p>),</span>
    <span class=n>outlier_in_non_hypermut_chisq_95</span><span class=p>)</span>

<span class=n>cgp_outlier_mutant_cases</span>
</code></pre></div> <blockquote> <p>"i_8228" "i_95FC" "i_BD58" "i_1C36" "i_502F" "i_D732" "i_6B06" "i_56B5"</p> </blockquote> <ul> <li>Pick random 9/73 cases<sup id=fnref:1><a class=footnote-ref href=#fn:1>1</a></sup> without outlier mutation profile.</li> </ul> <div class=highlight><pre><span></span><code><span class=c1>## nonhm_sampled &lt;- sample(ncol(fit_res_nonhm$contribution),</span>
<span class=c1>##     ncol(fit_res_hm$contribution), replace = FALSE)</span>

<span class=c1>## following cases were used in the right-side panel of fig 3A</span>
<span class=n>nonhm_sampled</span> <span class=o>&lt;-</span> <span class=nf>c</span><span class=p>(</span><span class=m>28</span><span class=p>,</span><span class=m>48</span><span class=p>,</span><span class=m>30</span><span class=p>,</span><span class=m>57</span><span class=p>,</span><span class=m>54</span><span class=p>,</span><span class=m>40</span><span class=p>,</span><span class=m>37</span><span class=p>,</span><span class=m>7</span><span class=p>,</span><span class=m>62</span><span class=p>)</span>
</code></pre></div> <ul> <li>Deconvolute known human mutational signatures</li> </ul> <blockquote> <p>Fitted mutational signatures, i.e., <code>fit_res</code>, <code>fit_res_hm</code> and <code>fit_res_nonhm</code> were derived using <code>fit_to_signatures</code> function from <em>MutationalPatterns</em> R package<sup id=fnref:2><a class=footnote-ref href=#fn:2>2</a></sup>.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>## mut_mat object is within /data/cgp_base_objects_v1.0.RData</span>
<span class=c1>## mut_mat &lt;- MutationalPattern::mut_matrix(vcf_list = vcfs,</span>
<span class=c1>##    ref_genome = ref_genome)</span>

<span class=n>fit_res</span> <span class=o>&lt;-</span> <span class=nf>fit_to_signatures</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>,</span> <span class=n>cancer_signatures</span><span class=p>)</span>

<span class=n>hypermutators</span> <span class=o>&lt;-</span> <span class=nf>c</span><span class=p>(</span><span class=s>&quot;i_8228&quot;</span><span class=p>,</span> <span class=s>&quot;i_95FC&quot;</span><span class=p>,</span> <span class=s>&quot;i_BD58&quot;</span><span class=p>,</span> <span class=s>&quot;i_1C36&quot;</span><span class=p>,</span>
    <span class=s>&quot;i_502F&quot;</span><span class=p>,</span> <span class=s>&quot;i_D732&quot;</span><span class=p>,</span> <span class=s>&quot;i_6B06&quot;</span><span class=p>,</span> <span class=s>&quot;i_56B5&quot;</span><span class=p>)</span>

<span class=n>non_hypermutators</span> <span class=o>&lt;-</span> <span class=nf>colnames</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>)[</span><span class=o>!</span><span class=nf>colnames</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>)</span> <span class=o>%in%</span> <span class=n>hypermutators</span><span class=p>]</span>

<span class=c1>## non-hypermutators or one without outlier mutation profile</span>
<span class=n>fit_res_nonhm</span> <span class=o>&lt;-</span> <span class=nf>fit_to_signatures</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>[,</span> <span class=nf>colnames</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>)</span> <span class=o>%in%</span>
    <span class=n>non_hypermutators</span><span class=p>],</span> <span class=n>cancer_signatures</span><span class=p>)</span>

<span class=c1>## hypermutators or one with outlier mutation profile</span>
<span class=n>fit_res_hm</span> <span class=o>&lt;-</span> <span class=nf>fit_to_signatures</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>[,</span> <span class=nf>colnames</span><span class=p>(</span><span class=n>mut_mat</span><span class=p>)</span> <span class=o>%in%</span>
    <span class=n>hypermutators</span><span class=p>],</span> <span class=n>cancer_signatures</span><span class=p>)</span>
</code></pre></div> <ul> <li>Merge signatures by proposed mechanism</li> </ul> <div class=highlight><pre><span></span><code><span class=n>hm_sigs</span> <span class=o>&lt;-</span> <span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span> <span class=nf>filter</span><span class=p>(</span><span class=n>sig_enrichment</span> <span class=o>!=</span> <span class=s>&quot;NON_HM&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>pull</span><span class=p>(</span><span class=n>sig_id</span><span class=p>)</span>

<span class=n>nonhm_sigs</span> <span class=o>&lt;-</span> <span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span> <span class=nf>filter</span><span class=p>(</span><span class=n>sig_enrichment</span> <span class=o>!=</span> <span class=s>&quot;HM&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>pull</span><span class=p>(</span><span class=n>sig_id</span><span class=p>)</span>

<span class=n>fit_res_hm_merged</span> <span class=o>&lt;-</span> <span class=nf>as_tibble</span><span class=p>(</span><span class=n>fit_res_hm</span><span class=o>$</span><span class=n>contribution</span><span class=p>[</span><span class=n>hm_sigs</span><span class=p>,</span> <span class=p>],</span>
    <span class=n>rownames</span> <span class=o>=</span> <span class=s>&quot;sig_id&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>left_join</span><span class=p>(</span><span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span> <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_id</span><span class=p>,</span> <span class=n>sig_group</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>patient_id</span><span class=p>,</span> <span class=n>sig_score</span><span class=p>,</span> <span class=nf>starts_with</span><span class=p>(</span><span class=s>&quot;i_&quot;</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>group_by</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>patient_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>group_score</span> <span class=o>=</span> <span class=nf>sum</span><span class=p>(</span><span class=n>sig_score</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>group_score</span><span class=p>,</span> <span class=n>patient_id</span><span class=p>,</span> <span class=n>sig_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>spread</span><span class=p>(</span><span class=n>patient_id</span><span class=p>,</span> <span class=n>group_score</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>ungroup</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=o>-</span><span class=n>sig_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>column_to_rownames</span><span class=p>(</span><span class=s>&quot;sig_group&quot;</span><span class=p>)</span>

<span class=n>fit_res_nonhm_merged</span> <span class=o>&lt;-</span> <span class=nf>as_tibble</span><span class=p>(</span><span class=n>fit_res_nonhm</span><span class=o>$</span><span class=n>contribution</span><span class=p>[</span><span class=n>nonhm_sigs</span><span class=p>,</span> <span class=p>],</span>
    <span class=n>rownames</span> <span class=o>=</span> <span class=s>&quot;sig_id&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>left_join</span><span class=p>(</span><span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span> <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_id</span><span class=p>,</span> <span class=n>sig_group</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>patient_id</span><span class=p>,</span> <span class=n>sig_score</span><span class=p>,</span> <span class=nf>starts_with</span><span class=p>(</span><span class=s>&quot;i_&quot;</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>group_by</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>patient_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>group_score</span> <span class=o>=</span> <span class=nf>sum</span><span class=p>(</span><span class=n>sig_score</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>group_score</span><span class=p>,</span> <span class=n>patient_id</span><span class=p>,</span> <span class=n>sig_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>spread</span><span class=p>(</span><span class=n>patient_id</span><span class=p>,</span> <span class=n>group_score</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>ungroup</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=o>-</span><span class=n>sig_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>column_to_rownames</span><span class=p>(</span><span class=s>&quot;sig_group&quot;</span><span class=p>)</span>

<span class=c1>## group cancer signatures context proportion for HM</span>

<span class=n>cancer_signatures_merged</span> <span class=o>&lt;-</span> <span class=nf>as_tibble</span><span class=p>(</span><span class=n>cancer_signatures</span><span class=p>,</span>
    <span class=n>rownames</span> <span class=o>=</span> <span class=s>&quot;context&quot;</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>gather</span><span class=p>(</span><span class=n>sig_id</span><span class=p>,</span> <span class=n>sig_score</span><span class=p>,</span> <span class=o>-</span><span class=n>context</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>left_join</span><span class=p>(</span><span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span> <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_id</span><span class=p>,</span> <span class=n>sig_group</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>filter</span><span class=p>(</span><span class=o>!</span><span class=nf>is.na</span><span class=p>(</span><span class=n>sig_group</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>group_by</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>context</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>mutate</span><span class=p>(</span><span class=n>group_score</span> <span class=o>=</span> <span class=nf>sum</span><span class=p>(</span><span class=n>sig_score</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>filter</span><span class=p>(</span><span class=o>!</span><span class=nf>is.na</span><span class=p>(</span><span class=n>group_score</span><span class=p>))</span> <span class=o>%&gt;%</span>
    <span class=nf>ungroup</span><span class=p>()</span> <span class=o>%&gt;%</span>
    <span class=n>dplyr</span><span class=o>::</span><span class=nf>select</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>group_score</span><span class=p>,</span> <span class=n>context</span><span class=p>,</span> <span class=n>sig_id</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>context</span><span class=p>,</span> <span class=n>group_score</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>spread</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>group_score</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>context</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>column_to_rownames</span><span class=p>(</span><span class=s>&quot;context&quot;</span><span class=p>)</span>

<span class=n>hm_nonhm_sig_group_colours</span> <span class=o>&lt;-</span> <span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>pull</span><span class=p>(</span><span class=n>colour_hex</span><span class=p>)</span>

<span class=nf>names</span><span class=p>(</span><span class=n>hm_nonhm_sig_group_colours</span><span class=p>)</span> <span class=o>&lt;-</span> <span class=n>hm_nonhm_topsigs_tbl</span> <span class=o>%&gt;%</span>
    <span class=nf>distinct</span><span class=p>(</span><span class=n>sig_group</span><span class=p>,</span> <span class=n>.keep_all</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span> <span class=o>%&gt;%</span>
    <span class=nf>pull</span><span class=p>(</span><span class=n>sig_group</span><span class=p>)</span>
</code></pre></div> <ul> <li>Function to plot signature contribution</li> </ul> <blockquote> <p>Rewrite of <code>plot_contribution</code> function from <em>MutationalPatterns</em> R package.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=n>cgp_plot_contribution</span> <span class=o>&lt;-</span> <span class=nf>function </span><span class=p>(</span><span class=n>contribution</span><span class=p>,</span>
    <span class=n>signatures</span><span class=p>,</span> <span class=n>index</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(),</span> <span class=n>coord_flip</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span>
    <span class=n>mode</span> <span class=o>=</span> <span class=s>&quot;relative&quot;</span><span class=p>,</span> <span class=n>palette</span> <span class=o>=</span> <span class=nf>c</span><span class=p>())</span>
    <span class=p>{</span>
        <span class=nf>if </span><span class=p>(</span><span class=o>!</span><span class=p>(</span><span class=n>mode</span> <span class=o>==</span> <span class=s>&quot;relative&quot;</span> <span class=o>|</span> <span class=n>mode</span> <span class=o>==</span> <span class=s>&quot;absolute&quot;</span><span class=p>))</span>
          <span class=nf>stop</span><span class=p>(</span><span class=s>&quot;mode parameter should be either &#39;relative&#39; or &#39;absolute&#39;&quot;</span><span class=p>)</span>

        <span class=nf>if </span><span class=p>(</span><span class=nf>length</span><span class=p>(</span><span class=n>index</span> <span class=o>&gt;</span> <span class=m>0</span><span class=p>))</span> <span class=p>{</span>
          <span class=n>contribution</span> <span class=o>=</span> <span class=n>contribution</span><span class=p>[,</span> <span class=n>index</span><span class=p>]</span>
        <span class=p>}</span>

        <span class=n>Sample</span> <span class=o>=</span> <span class=kc>NULL</span>
        <span class=n>Contribution</span> <span class=o>=</span> <span class=kc>NULL</span>
        <span class=n>Signature</span> <span class=o>=</span> <span class=kc>NULL</span>

        <span class=nf>if </span><span class=p>(</span><span class=n>mode</span> <span class=o>==</span> <span class=s>&quot;relative&quot;</span><span class=p>)</span> <span class=p>{</span>
            <span class=n>m_contribution</span> <span class=o>=</span> <span class=n>data.table</span><span class=o>::</span><span class=nf>melt</span><span class=p>(</span><span class=n>contribution</span><span class=p>,</span> <span class=s>&quot;sig_id&quot;</span><span class=p>)</span>
            <span class=nf>colnames</span><span class=p>(</span><span class=n>m_contribution</span><span class=p>)</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(</span><span class=s>&quot;Signature&quot;</span><span class=p>,</span>
                <span class=s>&quot;Sample&quot;</span><span class=p>,</span> <span class=s>&quot;Contribution&quot;</span><span class=p>)</span>

            <span class=n>plot</span> <span class=o>=</span> <span class=nf>ggplot</span><span class=p>(</span><span class=n>m_contribution</span><span class=p>,</span> <span class=nf>aes</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=nf>factor</span><span class=p>(</span><span class=n>Sample</span><span class=p>),</span>
              <span class=n>y</span> <span class=o>=</span> <span class=n>Contribution</span><span class=p>,</span> <span class=n>fill</span> <span class=o>=</span> <span class=nf>factor</span><span class=p>(</span><span class=n>Signature</span><span class=p>),</span> <span class=n>order</span> <span class=o>=</span> <span class=n>Sample</span><span class=p>))</span> <span class=o>+</span>
              <span class=nf>geom_bar</span><span class=p>(</span><span class=n>position</span> <span class=o>=</span> <span class=s>&quot;fill&quot;</span><span class=p>,</span> <span class=n>stat</span> <span class=o>=</span> <span class=s>&quot;identity&quot;</span><span class=p>,</span>
                <span class=n>colour</span> <span class=o>=</span> <span class=s>&quot;black&quot;</span><span class=p>)</span> <span class=o>+</span>
              <span class=nf>labs</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=s>&quot;&quot;</span><span class=p>,</span> <span class=n>y</span> <span class=o>=</span> <span class=s>&quot;Relative contribution&quot;</span><span class=p>)</span> <span class=o>+</span> <span class=nf>theme_bw</span><span class=p>()</span> <span class=o>+</span>
              <span class=nf>theme</span><span class=p>(</span><span class=n>panel.grid.minor.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
                <span class=n>panel.grid.major.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>())</span> <span class=o>+</span>
              <span class=nf>theme</span><span class=p>(</span><span class=n>panel.grid.minor.y</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
                <span class=n>panel.grid.major.y</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>())</span>
        <span class=p>}</span> <span class=n>else</span> <span class=p>{</span>
            <span class=nf>if </span><span class=p>(</span><span class=nf>missing</span><span class=p>(</span><span class=n>signatures</span><span class=p>))</span>
              <span class=nf>stop</span><span class=p>(</span><span class=nf>paste</span><span class=p>(</span><span class=s>&quot;For contribution plotting in mode &#39;absolute&#39;:&quot;</span><span class=p>,</span>
                  <span class=s>&quot;also provide signatures matrix&quot;</span><span class=p>))</span>

            <span class=n>total_signatures</span> <span class=o>=</span> <span class=nf>colSums</span><span class=p>(</span><span class=n>signatures</span><span class=p>)</span>
            <span class=n>abs_contribution</span> <span class=o>=</span> <span class=n>contribution</span><span class=p>[,</span><span class=m>-1</span><span class=p>]</span> <span class=o>*</span> <span class=n>total_signatures</span>
            <span class=nf>rownames</span><span class=p>(</span><span class=n>abs_contribution</span><span class=p>)</span> <span class=o>=</span> <span class=n>contribution</span> <span class=o>%&gt;%</span>
                <span class=nf>pull</span><span class=p>(</span><span class=n>sig_id</span><span class=p>)</span>

            <span class=n>m_contribution</span> <span class=o>=</span> <span class=n>data.table</span><span class=o>::</span><span class=nf>melt</span><span class=p>(</span><span class=n>abs_contribution</span><span class=p>,</span> <span class=s>&quot;sig_id&quot;</span><span class=p>)</span>
            <span class=nf>colnames</span><span class=p>(</span><span class=n>m_contribution</span><span class=p>)</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(</span><span class=s>&quot;Signature&quot;</span><span class=p>,</span>
                <span class=s>&quot;Sample&quot;</span><span class=p>,</span> <span class=s>&quot;Contribution&quot;</span><span class=p>)</span>

            <span class=n>plot</span> <span class=o>=</span> <span class=nf>ggplot</span><span class=p>(</span><span class=n>m_contribution</span><span class=p>,</span> <span class=nf>aes</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=nf>factor</span><span class=p>(</span><span class=n>Sample</span><span class=p>),</span>
              <span class=n>y</span> <span class=o>=</span> <span class=n>Contribution</span><span class=p>,</span> <span class=n>fill</span> <span class=o>=</span> <span class=nf>factor</span><span class=p>(</span><span class=n>Signature</span><span class=p>),</span> <span class=n>order</span> <span class=o>=</span> <span class=n>Sample</span><span class=p>))</span> <span class=o>+</span>
              <span class=nf>geom_bar</span><span class=p>(</span><span class=n>stat</span> <span class=o>=</span> <span class=s>&quot;identity&quot;</span><span class=p>,</span> <span class=n>colour</span> <span class=o>=</span> <span class=s>&quot;black&quot;</span><span class=p>)</span> <span class=o>+</span>
              <span class=nf>labs</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=s>&quot;&quot;</span><span class=p>,</span> <span class=n>y</span> <span class=o>=</span> <span class=s>&quot;Absolute contribution \n (no. mutations)&quot;</span><span class=p>)</span> <span class=o>+</span>
              <span class=nf>theme_bw</span><span class=p>()</span> <span class=o>+</span>
              <span class=nf>theme</span><span class=p>(</span><span class=n>panel.grid.minor.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
                <span class=n>panel.grid.major.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>())</span> <span class=o>+</span>
              <span class=nf>theme</span><span class=p>(</span><span class=n>panel.grid.minor.y</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
                <span class=n>panel.grid.major.y</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>())</span>
        <span class=p>}</span>

        <span class=nf>if </span><span class=p>(</span><span class=nf>length</span><span class=p>(</span><span class=n>palette</span><span class=p>)</span> <span class=o>&gt;</span> <span class=m>0</span><span class=p>)</span> <span class=p>{</span>
            <span class=n>plot</span> <span class=o>=</span> <span class=n>plot</span> <span class=o>+</span>
                <span class=nf>scale_fill_manual</span><span class=p>(</span><span class=n>name</span> <span class=o>=</span> <span class=s>&quot;Signature&quot;</span><span class=p>,</span> <span class=n>values</span> <span class=o>=</span> <span class=n>palette</span><span class=p>)</span>
        <span class=p>}</span> <span class=n>else</span> <span class=p>{</span>
            <span class=n>plot</span> <span class=o>=</span> <span class=n>plot</span> <span class=o>+</span>
                <span class=nf>scale_fill_discrete</span><span class=p>(</span><span class=n>name</span> <span class=o>=</span> <span class=s>&quot;Signature&quot;</span><span class=p>)</span>
        <span class=p>}</span>

        <span class=nf>if </span><span class=p>(</span><span class=n>coord_flip</span><span class=p>)</span> <span class=p>{</span>
            <span class=n>plot</span> <span class=o>=</span> <span class=n>plot</span> <span class=o>+</span>
                <span class=nf>coord_flip</span><span class=p>()</span> <span class=o>+</span>
                <span class=nf>xlim</span><span class=p>(</span><span class=nf>rev</span><span class=p>(</span><span class=nf>levels</span><span class=p>(</span><span class=nf>factor</span><span class=p>(</span><span class=n>m_contribution</span><span class=o>$</span><span class=n>Sample</span><span class=p>))))</span>
        <span class=p>}</span> <span class=n>else</span> <span class=p>{</span>
            <span class=n>plot</span> <span class=o>=</span> <span class=n>plot</span> <span class=o>+</span> <span class=nf>xlim</span><span class=p>(</span><span class=nf>levels</span><span class=p>(</span><span class=nf>factor</span><span class=p>(</span><span class=n>m_contribution</span><span class=o>$</span><span class=n>Sample</span><span class=p>)))</span>
        <span class=p>}</span>

        <span class=nf>return</span><span class=p>(</span><span class=n>plot</span><span class=p>)</span>
    <span class=p>}</span>
</code></pre></div> <h3 id=fig-3a>Fig 3A<a class=headerlink href=#fig-3a title="Permanent link">&para;</a></h3> <blockquote> <p>Plot deconvoluted signature contribution after grouping signatures based on proposed mechanism.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>## plot canine glioma patients with outlier mutation profile</span>

<span class=n>plotsig_hm_rel_merged</span> <span class=o>&lt;-</span> <span class=nf>cgp_plot_contribution</span><span class=p>(</span><span class=nf>as_tibble</span><span class=p>(</span>
    <span class=n>fit_res_hm_merged</span><span class=p>,</span> <span class=n>rownames</span> <span class=o>=</span> <span class=s>&quot;sig_id&quot;</span><span class=p>),</span>
    <span class=n>cancer_signatures_merged</span><span class=p>,</span>
    <span class=n>coord_flip</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span>
    <span class=n>mode</span> <span class=o>=</span> <span class=s>&quot;relative&quot;</span><span class=p>,</span> <span class=n>palette</span> <span class=o>=</span> <span class=n>hm_nonhm_sig_group_colours</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme_cowplot</span><span class=p>(</span><span class=n>font_size</span> <span class=o>=</span> <span class=m>20</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>axis.text.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
        <span class=n>legend.position</span> <span class=o>=</span> <span class=s>&quot;bottom&quot;</span><span class=p>,</span>
        <span class=n>legend.text</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>14</span><span class=p>))</span> <span class=o>+</span>
    <span class=nf>labs</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=s>&quot;Canine Gliomas with outlier mutation profile\n(n=8)&quot;</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>guides</span><span class=p>(</span><span class=n>fill</span> <span class=o>=</span> <span class=nf>guide_legend</span><span class=p>(</span>
        <span class=n>title</span> <span class=o>=</span> <span class=s>&quot;Known signatures grouped by\ntheir proposed mechanism&quot;</span><span class=p>,</span>
    <span class=n>title.position</span> <span class=o>=</span> <span class=s>&quot;bottom&quot;</span><span class=p>,</span>
    <span class=n>label.theme</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>14</span><span class=p>),</span>
    <span class=n>keywidth</span><span class=o>=</span><span class=m>0.8</span><span class=p>,</span>
    <span class=n>keyheight</span><span class=o>=</span><span class=m>0.8</span><span class=p>,</span>
    <span class=n>default.unit</span><span class=o>=</span><span class=s>&quot;cm&quot;</span><span class=p>,</span> <span class=n>nrow</span> <span class=o>=</span> <span class=m>2</span><span class=p>,</span><span class=n>byrow</span><span class=o>=</span><span class=kc>TRUE</span><span class=p>))</span>

<span class=c1>## plot randomly sampled 9 canine patients without outlier mutation profile</span>

<span class=n>plotsig_nonhm_rel_sampled_merged</span> <span class=o>&lt;-</span> <span class=nf>cgp_plot_contribution</span><span class=p>(</span><span class=nf>as_tibble</span><span class=p>(</span>
    <span class=n>fit_res_nonhm_merged</span><span class=p>[</span> <span class=p>,</span><span class=n>nonhm_sampled</span><span class=p>],</span> <span class=n>rownames</span> <span class=o>=</span> <span class=s>&quot;sig_id&quot;</span><span class=p>),</span>
    <span class=n>cancer_signatures_merged</span><span class=p>,</span>
    <span class=n>coord_flip</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span>
    <span class=n>mode</span> <span class=o>=</span> <span class=s>&quot;relative&quot;</span><span class=p>,</span> <span class=n>palette</span> <span class=o>=</span> <span class=n>hm_nonhm_sig_group_colours</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme_cowplot</span><span class=p>(</span><span class=n>font_size</span> <span class=o>=</span> <span class=m>20</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>axis.text.x</span> <span class=o>=</span> <span class=nf>element_blank</span><span class=p>(),</span>
        <span class=n>legend.position</span> <span class=o>=</span> <span class=s>&quot;bottom&quot;</span><span class=p>,</span>
        <span class=n>legend.text</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>14</span><span class=p>))</span> <span class=o>+</span>
        <span class=nf>labs</span><span class=p>(</span><span class=n>x</span> <span class=o>=</span> <span class=s>&quot;Canine Gliomas with no outlier mutation profile\n(9/73)&quot;</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>guides</span><span class=p>(</span><span class=n>fill</span> <span class=o>=</span> <span class=nf>guide_legend</span><span class=p>(</span>
        <span class=n>title</span> <span class=o>=</span> <span class=s>&quot;Signatures grouped by\ntheir proposed mechanism&quot;</span><span class=p>,</span>
    <span class=n>title.position</span> <span class=o>=</span> <span class=s>&quot;bottom&quot;</span><span class=p>,</span>
    <span class=n>label.theme</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>14</span><span class=p>),</span>
    <span class=n>keywidth</span><span class=o>=</span><span class=m>0.8</span><span class=p>,</span>
    <span class=n>keyheight</span><span class=o>=</span><span class=m>0.8</span><span class=p>,</span>
    <span class=n>default.unit</span><span class=o>=</span><span class=s>&quot;cm&quot;</span><span class=p>,</span> <span class=n>nrow</span> <span class=o>=</span> <span class=m>2</span><span class=p>,</span><span class=n>byrow</span><span class=o>=</span><span class=kc>TRUE</span><span class=p>))</span>

<span class=nf>ggsave</span><span class=p>(</span><span class=s>&quot;F3A.pdf&quot;</span><span class=p>,</span>
    <span class=nf>grid.arrange</span><span class=p>(</span><span class=n>plotsig_hm_rel_merged</span><span class=p>,</span>
                 <span class=n>plotsig_nonhm_rel_sampled_merged</span><span class=p>,</span>
        <span class=n>ncol</span> <span class=o>=</span> <span class=m>2</span><span class=p>,</span> <span class=n>widths</span> <span class=o>=</span> <span class=nf>c</span><span class=p>(</span><span class=m>5</span><span class=p>,</span> <span class=m>5</span><span class=p>)),</span>
        <span class=n>width</span> <span class=o>=</span> <span class=m>18</span><span class=p>,</span>
        <span class=n>height</span> <span class=o>=</span> <span class=m>12</span><span class=p>,</span>
        <span class=n>useDingbats</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>
</code></pre></div> <h3 id=suppl-fig-3a>Suppl Fig 3A<a class=headerlink href=#suppl-fig-3a title="Permanent link">&para;</a></h3> <blockquote> <p>Plot absolute and relative contribution of dominant signatures.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=c1>## Select signatures with major contribution &gt; 3rd Qu (2.75)</span>
<span class=nf>summary</span><span class=p>(</span><span class=nf>rowMedians</span><span class=p>(</span><span class=n>fit_res</span><span class=o>$</span><span class=n>contribution</span><span class=p>))</span>

<span class=n>select_high</span> <span class=o>&lt;-</span> <span class=nf>rownames</span><span class=p>(</span><span class=n>fit_res</span><span class=o>$</span><span class=n>contribution</span><span class=p>)[</span>
    <span class=nf>which</span><span class=p>(</span><span class=nf>rowMedians</span><span class=p>(</span><span class=n>fit_res</span><span class=o>$</span><span class=n>contribution</span><span class=p>)</span> <span class=o>&gt;</span> <span class=m>2.75</span><span class=p>)</span>
    <span class=p>]</span>

<span class=n>select_high</span>

<span class=n>color_lib</span> <span class=o>&lt;-</span> <span class=nf>c</span><span class=p>(</span><span class=s>&#39;#8e0152&#39;</span><span class=p>,</span><span class=s>&#39;#c51b7d&#39;</span><span class=p>,</span><span class=s>&#39;#de77ae&#39;</span><span class=p>,</span><span class=s>&#39;#f1b6da&#39;</span><span class=p>,</span><span class=s>&#39;#fde0ef&#39;</span><span class=p>,</span>
  <span class=s>&#39;#f7f7f7&#39;</span><span class=p>,</span><span class=s>&#39;#e6f5d0&#39;</span><span class=p>,</span><span class=s>&#39;#b8e186&#39;</span><span class=p>,</span><span class=s>&#39;#7fbc41&#39;</span><span class=p>,</span><span class=s>&#39;#4d9221&#39;</span><span class=p>,</span>
  <span class=s>&#39;#276419&#39;</span><span class=p>)</span>

<span class=c1>## Plot contribution</span>
<span class=nf>ggsave</span><span class=p>(</span><span class=s>&quot;SF3A_top_panel.pdf&quot;</span><span class=p>,</span>
    <span class=nf>plot_contribution</span><span class=p>(</span><span class=n>fit_res</span><span class=o>$</span><span class=n>contribution</span><span class=p>[</span><span class=n>select_high</span><span class=p>,</span> <span class=p>],</span>
         <span class=n>cancer_signatures</span><span class=p>[</span> <span class=p>,</span> <span class=n>select_high</span><span class=p>],</span>
         <span class=n>coord_flip</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span>
         <span class=n>mode</span> <span class=o>=</span> <span class=s>&quot;absolute&quot;</span><span class=p>,</span>
         <span class=n>palette</span> <span class=o>=</span> <span class=n>color_lib</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>axis.text.x</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>angle</span> <span class=o>=</span> <span class=m>90</span><span class=p>)),</span>
    <span class=n>width</span> <span class=o>=</span> <span class=m>9</span><span class=p>,</span>
    <span class=n>height</span> <span class=o>=</span> <span class=m>5</span><span class=p>,</span>
    <span class=n>useDingbats</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>

<span class=nf>ggsave</span><span class=p>(</span><span class=s>&quot;SF3A_bottom_panel.pdf&quot;</span><span class=p>,</span>
    <span class=nf>plot_contribution</span><span class=p>(</span><span class=n>fit_res</span><span class=o>$</span><span class=n>contribution</span><span class=p>[</span><span class=n>select_high</span><span class=p>,</span> <span class=p>],</span>
        <span class=n>cancer_signatures</span><span class=p>[</span> <span class=p>,</span> <span class=n>select_high</span><span class=p>],</span>
        <span class=n>coord_flip</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span>
        <span class=n>mode</span> <span class=o>=</span> <span class=s>&quot;relative&quot;</span><span class=p>,</span>
        <span class=n>palette</span> <span class=o>=</span> <span class=n>color_lib</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>axis.text.x</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>angle</span> <span class=o>=</span> <span class=m>90</span><span class=p>)),</span>
    <span class=n>width</span> <span class=o>=</span> <span class=m>9</span><span class=p>,</span>
    <span class=n>height</span> <span class=o>=</span> <span class=m>5</span><span class=p>,</span>
    <span class=n>useDingbats</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>
</code></pre></div> <h3 id=suppl-fig-3b>Suppl Fig 3B<a class=headerlink href=#suppl-fig-3b title="Permanent link">&para;</a></h3> <blockquote> <p>Boxplot comparison of genomic instability related signatures between canine glioma patients separated by outlier mutation profile.</p> </blockquote> <div class=highlight><pre><span></span><code><span class=n>suppl_fig_3B</span> <span class=o>&lt;-</span> <span class=n>ggpubr</span><span class=o>::</span><span class=nf>ggboxplot</span><span class=p>(</span><span class=n>instability_sigs_df</span><span class=p>,</span>
        <span class=n>x</span> <span class=o>=</span> <span class=s>&quot;hypermut&quot;</span><span class=p>,</span> <span class=n>y</span> <span class=o>=</span> <span class=s>&quot;sig_contrib&quot;</span><span class=p>,</span>
        <span class=n>add</span> <span class=o>=</span> <span class=s>&quot;jitter&quot;</span><span class=p>,</span>
        <span class=n>facet.by</span> <span class=o>=</span> <span class=s>&quot;sig_group&quot;</span><span class=p>,</span>
        <span class=n>scales</span> <span class=o>=</span> <span class=s>&quot;free_y&quot;</span><span class=p>,</span>
        <span class=n>bxp.errorbar</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>,</span> <span class=n>notch</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>,</span>
        <span class=n>font.label</span> <span class=o>=</span> <span class=nf>list</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>7</span><span class=p>,</span> <span class=n>color</span> <span class=o>=</span> <span class=s>&quot;black&quot;</span><span class=p>),</span>
        <span class=n>ylab</span> <span class=o>=</span> <span class=s>&quot;Signature contribution&quot;</span><span class=p>,</span>
        <span class=n>xlab</span> <span class=o>=</span> <span class=s>&quot;Canine Gliomas separated by\noutlier mutation profile&quot;</span><span class=p>)</span> <span class=o>+</span>
    <span class=nf>theme</span><span class=p>(</span><span class=n>axis.text</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>11</span><span class=p>),</span>
        <span class=n>axis.title</span> <span class=o>=</span> <span class=nf>element_text</span><span class=p>(</span><span class=n>size</span> <span class=o>=</span> <span class=m>11</span><span class=p>))</span> <span class=o>+</span>
    <span class=n>ggpubr</span><span class=o>::</span><span class=nf>stat_compare_means</span><span class=p>(</span><span class=n>method</span> <span class=o>=</span> <span class=s>&quot;wilcox.test&quot;</span><span class=p>,</span>
    <span class=n>label.y.npc</span> <span class=o>=</span> <span class=m>0.9</span><span class=p>,</span> <span class=n>na.rm</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>)</span>

<span class=nf>ggsave</span><span class=p>(</span><span class=s>&quot;SF3B.pdf&quot;</span><span class=p>,</span>
       <span class=n>suppl_fig_3B</span><span class=p>,</span>
       <span class=n>width</span><span class=o>=</span><span class=m>12</span><span class=p>,</span>
       <span class=n>height</span><span class=o>=</span><span class=m>8</span><span class=p>,</span>
       <span class=n>useDingbats</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>
</code></pre></div> <div class=footnote> <hr> <ol> <li id=fn:1> <p>Earlier version of <code>fit_res_hm</code> had 9 and not 8 canine glioma patients with outlier mutation profile. Therefore, 9 cases were selected at random from remaining patients without outlier mutation profile.&#160;<a class=footnote-backref href=#fnref:1 title="Jump back to footnote 1 in the text">&#8617;</a></p> </li> <li id=fn:2> <p>Blokzijl F, Janssen R, van Boxtel R, Cuppen E. MutationalPatterns: comprehensive genome-wide analysis of mutational processes. Genome Med 2018;10:33. doi: <a href=https://doi.org/10.1186/s13073-018-0539-0>10.1186/s13073-018-0539-0</a>.&#160;<a class=footnote-backref href=#fnref:2 title="Jump back to footnote 2 in the text">&#8617;</a></p> </li> </ol> </div> <!-- Source file information --> <hr> <div class=md-source-file> <small> Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-iso_date">2020-02-07</span> </small> </div> <!-- end default contents --> <!-- Get setting from mkdocs.yml, but allow page-level overrides --> <!-- Inject Disqus into current page --> <p><small>Author: <a href=https://sbamin.com/about rel=me>Samir B. Amin</a> <span class=twemoji> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg></span><a href=https://twitter.com/sbamin>@sbamin</a></small></p> <!-- Giscus --> <h2 id=__comments>Comments</h2> <p>Comments are powered by <span class=twemoji><svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 16 16"><path fill-rule=evenodd d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg></span> <a href=https://github.com/TheJacksonLaboratory/canineglioma/discussions>github discussions</a></p> <!-- Replace with generated snippet --> <script src=https://giscus.app/client.js data-repo=TheJacksonLaboratory/canineglioma data-repo-id="MDEwOlJlcG9zaXRvcnkxOTEyOTczMjg=" data-category=webchat data-category-id=DIC_kwDOC2b3MM4CBGtB data-mapping=pathname data-reactions-enabled=1 data-emit-metadata=0 data-input-position=top data-theme=light data-lang=en crossorigin=anonymous async>
	</script> <!-- Reload on palette change --> <script>
	var palette = __md_get("__palette")
	if (palette && typeof palette.color === "object")
	  if (palette.color.scheme === "slate") {
	    var giscus = document.querySelector("script[src*=giscus]")
	    giscus.setAttribute("data-theme", "dark") 
	  }

	/* Register event handlers after documented loaded */
	document.addEventListener("DOMContentLoaded", function() {
	  var ref = document.querySelector("[data-md-component=palette]")
	  ref.addEventListener("change", function() {
	    var palette = __md_get("__palette")
	    if (palette && typeof palette.color === "object") {
	      var theme = palette.color.scheme === "slate" ? "dark" : "light"

	      /* Instruct Giscus to change theme */
	      var frame = document.querySelector(".giscus-frame")
	      frame.contentWindow.postMessage(
	        { giscus: { setConfig: { theme } } },
	        "https://giscus.app"
	      )
	    }
	  })
	})
	</script> </article> </div> </div> <a href=# class="md-top md-icon" data-md-component=top data-md-state=hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12z"/></svg> Back to top </a> </main> <!-- Footer --> <footer class=md-footer> <!-- Link to previous and/or next page --> <nav class="md-footer__inner md-grid" aria-label=Footer> <!-- Link to previous page --> <a href=../S2/ class="md-footer__link md-footer__link--prev" aria-label="Previous: S2 &middot; IGV Plots" rel=prev> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </div> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Previous </span> S2 &middot; IGV Plots </div> </div> </a> <!-- Link to next page --> <a href=../F3B/ class="md-footer__link md-footer__link--next" aria-label="Next: F3B &middot; Somatic Mutational Landscape - de novo" rel=next> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Next </span> F3B &middot; Somatic Mutational Landscape - de novo </div> </div> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg> </div> </a> </nav> <!-- Further information --> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <!-- Copyright and theme information --> <div class=md-copyright> <div class=md-copyright__highlight> &copy; 2010-2022: <a href=https://verhaaklab.com title=Home>Verhaak Lab</a> | <a href=https://verhaaklab.com/privacy/ >Disclosure</a> </div> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> <!-- Social links --> <div class=md-social> <a href=https://verhaaklab.com/contact target=_blank rel=noopener title=verhaaklab.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M448 64H64C28.65 64 0 92.65 0 128v256c0 35.35 28.65 64 64 64h384c35.35 0 64-28.65 64-64V128c0-35.35-28.7-64-64-64zM64 112h384c8.822 0 16 7.178 16 16v22.16l-166.8 138.1c-23.19 19.28-59.34 19.27-82.47.016L48 150.2V128c0-8.8 7.18-16 16-16zm384 288H64c-8.822 0-16-7.178-16-16V212.7l136.1 113.4c20.2 16.7 45.7 25.9 71.9 25.9s51.75-9.188 71.97-25.98L464 212.7V384c0 8.8-7.2 16-16 16z"/></svg> </a> <a href=https://twitter.com/roelverhaak target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> <a href=https://github.com/sbamin target=_blank rel=noopener title=github.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 496 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg> </a> <a href=https://twitter.com/sbamin target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><!-- Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "../..", "features": ["navigation.tabs", "navigation.instant", "navigation.top", "navigation.indexes", "navigation.sections", "content.code.annotate"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../assets/javascripts/workers/search.bd0b6b67.min.js", "version": {"provider": "mike", "default": "latest"}}</script> <script src=../../assets/javascripts/bundle.467223ff.min.js></script> <script src=../../assets/js/extra.js></script> <script src=../../assets/js/mathjax.js></script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js></script> <script src=https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js></script> <script src=../../assets/js/tables.js></script> </body> </html>