<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=author content=sbamin><link href=https://canineglioma.verhaaklab.com/run_code/run_all/ rel=canonical><link rel=icon href=../../assets/images/fixed/favicon.png><meta name=generator content="mkdocs-1.2.3, mkdocs-material-8.1.2"><title>cgp_figures - Canine Glioma Project &middot; Verhaak Lab</title><link rel=stylesheet href=../../assets/stylesheets/main.f7951f6f.min.css><link rel=stylesheet href=../../assets/stylesheets/palette.e6a45f82.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CFira+Code&display=fallback"><style>:root{--md-text-font:"Roboto";--md-code-font:"Fira Code"}</style><link rel=stylesheet href=../../assets/css/extra.css><script>__md_scope=new URL("../..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script><!-- Determine analytics provider (deprecated, removed in v8) --><!-- Determine analytics provider --><!-- Set up analytics provider --><!-- Determine analytics property (deprecated, removed in v8) --><!-- Determine analytics property --><!-- Google Analytics 4 (G-XXXXXXXXXX) --><script>
    window.ga = window.ga || function() {
      (ga.q = ga.q || []).push(arguments)
    }
    ga.l = +new Date()

    /* Set up integration and send page view */
    ga("create", "UA-64053990-2", "auto")
    ga("set", "anonymizeIp", true)
    ga("send", "pageview")

    /* Register virtual event handlers */
    document.addEventListener("DOMContentLoaded", function() {
      if (document.forms.search) {
        var query = document.forms.search.query
        query.addEventListener("blur", function() {
          if (this.value) {
            var path = document.location.pathname;
            ga("send", "pageview", path + "?q=" + this.value)
          }
        })
      }

      /* Send page view on location change */
      if (typeof location$ !== "undefined")
        location$.subscribe(function(url) {
          ga("send", "pageview", url.pathname)
        })
    })
  </script><script async src=https://www.google-analytics.com/analytics.js></script><!--   <link rel="stylesheet" href="https://canineglioma.verhaaklab.com/../../assets/stylesheets/main.c296cc61.min.css"> --><meta name=keywords content="canine,glioma,pediatric,comparative genomics,DIPG,verhaak,lab,jax"><meta property=og:type content=website><meta property=og:title content="Output · Canine Glioma Project · Verhaak Lab"><meta property=og:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta content=https://canineglioma.verhaaklab.com/run_code/run_all/ property=og:url><meta property=og:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta property=og:image:type content=image/png><meta property=og:image:width content=256><meta property=og:image:height content=256><meta name=twitter:card content=summary><meta name=twitter:site content=@sbamin><meta name=twitter:creator content=@sbamin><meta name=twitter:title content="Output · Canine Glioma Project · Verhaak Lab"><meta name=twitter:description content="Canine Glioma Project · Extended Methods, Data, and Code for Figures"><meta name=twitter:image content=https://canineglioma.verhaaklab.com/assets/images/fixed/sitelogo.png><meta name=twitter:image:alt content="webpage banner or site logo"></head> <body dir=ltr data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo> <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#comparative-molecular-life-history-of-spontaneous-canine-and-human-gliomas class=md-skip> Skip to content </a> </div> <div data-md-component=announce> </div> <div data-md-component=outdated hidden> <aside class="md-banner md-banner--warning"> </aside> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-header__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> Canine Glioma Project &middot; Verhaak Lab </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> cgp_figures </span> </div> </div> </div> <form class=md-header__option data-md-component=palette> <input class=md-option data-md-color-media data-md-color-scheme=default data-md-color-primary=blue data-md-color-accent=indigo aria-label="Switch to dark mode" type=radio name=__palette id=__palette_1> <label class="md-header__button md-icon" title="Switch to dark mode" for=__palette_2 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg> </label> <input class=md-option data-md-color-media data-md-color-scheme=slate data-md-color-primary=teal data-md-color-accent=light-green aria-label="Switch to light mode" type=radio name=__palette id=__palette_2> <label class="md-header__button md-icon" title="Switch to light mode" for=__palette_1 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3z"/></svg> </label> </form> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </label> <nav class=md-search__options aria-label=Search> <button type=reset class="md-search__icon md-icon" aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg> </button> </nav> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M100.59 334.24c48.57 3.31 58.95 2.11 58.95 11.94 0 20-65.55 20.06-65.55 1.52.01-5.09 3.29-9.4 6.6-13.46zm27.95-116.64c-32.29 0-33.75 44.47-.75 44.47 32.51 0 31.71-44.47.75-44.47zM448 80v352a48 48 0 0 1-48 48H48a48 48 0 0 1-48-48V80a48 48 0 0 1 48-48h352a48 48 0 0 1 48 48zm-227 69.31c0 14.49 8.38 22.88 22.86 22.88 14.74 0 23.13-8.39 23.13-22.88S258.62 127 243.88 127c-14.48 0-22.88 7.84-22.88 22.31zM199.18 195h-49.55c-25-6.55-81.56-4.85-81.56 46.75 0 18.8 9.4 32 21.85 38.11C74.23 294.23 66.8 301 66.8 310.6c0 6.87 2.79 13.22 11.18 16.76-8.9 8.4-14 14.48-14 25.92C64 373.35 81.53 385 127.52 385c44.22 0 69.87-16.51 69.87-45.73 0-36.67-28.23-35.32-94.77-39.38l8.38-13.43c17 4.74 74.19 6.23 74.19-42.43 0-11.69-4.83-19.82-9.4-25.67l23.38-1.78zm84.34 109.84-13-1.78c-3.82-.51-4.07-1-4.07-5.09V192.52h-52.6l-2.79 20.57c15.75 5.55 17 4.86 17 10.17V298c0 5.62-.31 4.58-17 6.87v20.06h72.42zM384 315l-6.87-22.37c-40.93 15.37-37.85-12.41-37.85-16.73v-60.72h37.85v-25.41h-35.82c-2.87 0-2 2.52-2-38.63h-24.18c-2.79 27.7-11.68 38.88-34 41.42v22.62c20.47 0 19.82-.85 19.82 2.54v66.57c0 28.72 11.43 40.91 41.67 40.91 14.45 0 30.45-4.83 41.38-10.2z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class="md-tabs__inner md-grid"> <ul class=md-tabs__list> <li class=md-tabs__item> <a href=../.. class=md-tabs__link> Home </a> </li> <li class=md-tabs__item> <a href=../../methods/S01_samples/ class=md-tabs__link> Methods </a> </li> <li class=md-tabs__item> <a href=../../figures/A1_preload/ class=md-tabs__link> Figures </a> </li> <li class=md-tabs__item> <a href=../ class="md-tabs__link md-tabs__link--active"> R Markdown </a> </li> <li class=md-tabs__item> <a href=https://verhaaklab.com/contact class=md-tabs__link> Contact </a> </li> </ul> </div> </nav> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=../.. title="Canine Glioma Project &middot; Verhaak Lab" class="md-nav__button md-logo" aria-label="Canine Glioma Project · Verhaak Lab" data-md-component=logo> <img src=../../assets/images/fixed/favicon.png alt=logo> </a> Canine Glioma Project &middot; Verhaak Lab </label> <div class=md-nav__source> <a href=https://github.com/TheJacksonLaboratory/canineglioma title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M100.59 334.24c48.57 3.31 58.95 2.11 58.95 11.94 0 20-65.55 20.06-65.55 1.52.01-5.09 3.29-9.4 6.6-13.46zm27.95-116.64c-32.29 0-33.75 44.47-.75 44.47 32.51 0 31.71-44.47.75-44.47zM448 80v352a48 48 0 0 1-48 48H48a48 48 0 0 1-48-48V80a48 48 0 0 1 48-48h352a48 48 0 0 1 48 48zm-227 69.31c0 14.49 8.38 22.88 22.86 22.88 14.74 0 23.13-8.39 23.13-22.88S258.62 127 243.88 127c-14.48 0-22.88 7.84-22.88 22.31zM199.18 195h-49.55c-25-6.55-81.56-4.85-81.56 46.75 0 18.8 9.4 32 21.85 38.11C74.23 294.23 66.8 301 66.8 310.6c0 6.87 2.79 13.22 11.18 16.76-8.9 8.4-14 14.48-14 25.92C64 373.35 81.53 385 127.52 385c44.22 0 69.87-16.51 69.87-45.73 0-36.67-28.23-35.32-94.77-39.38l8.38-13.43c17 4.74 74.19 6.23 74.19-42.43 0-11.69-4.83-19.82-9.4-25.67l23.38-1.78zm84.34 109.84-13-1.78c-3.82-.51-4.07-1-4.07-5.09V192.52h-52.6l-2.79 20.57c15.75 5.55 17 4.86 17 10.17V298c0 5.62-.31 4.58-17 6.87v20.06h72.42zM384 315l-6.87-22.37c-40.93 15.37-37.85-12.41-37.85-16.73v-60.72h37.85v-25.41h-35.82c-2.87 0-2 2.52-2-38.63h-24.18c-2.79 27.7-11.68 38.88-34 41.42v22.62c20.47 0 19.82-.85 19.82 2.54v66.57c0 28.72 11.43 40.91 41.67 40.91 14.45 0 30.45-4.83 41.38-10.2z"/></svg> </div> <div class=md-source__repository> TheJacksonLaboratory/canineglioma </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../.. class=md-nav__link> Home </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_2 type=checkbox id=__nav_2> <label class=md-nav__link for=__nav_2> Methods <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Methods data-md-level=1> <label class=md-nav__title for=__nav_2> <span class="md-nav__icon md-icon"></span> Methods </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../methods/S01_samples/ class=md-nav__link> Samples </a> </li> <li class=md-nav__item> <a href=../../methods/S02_data_sources/ class=md-nav__link> Data Sources </a> </li> <li class=md-nav__item> <a href=../../methods/S03_sample_prep/ class=md-nav__link> Sample Preparations </a> </li> <li class=md-nav__item> <a href=../../methods/S04_alignments_fprint/ class=md-nav__link> Alignments </a> </li> <li class=md-nav__item> <a href=../../methods/S05_somatic_variants/ class=md-nav__link> Somatic SNVs and Indels </a> </li> <li class=md-nav__item> <a href=../../methods/S06_smgs/ class=md-nav__link> Significantly Mutated Genes </a> </li> <li class=md-nav__item> <a href=../../methods/S07_hallmarks/ class=md-nav__link> Cancer Hallmarks </a> </li> <li class=md-nav__item> <a href=../../methods/S09_mutation_rate/ class=md-nav__link> Somatic Mutation Rate </a> </li> <li class=md-nav__item> <a href=../../methods/S10_SCNA/ class=md-nav__link> Somatic Copy Number Aberrations </a> </li> <li class=md-nav__item> <a href=../../methods/S11_aneuploidy/ class=md-nav__link> Aneuploidy Metrics </a> </li> <li class=md-nav__item> <a href=../../methods/S12_ith/ class=md-nav__link> Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../../methods/S13_mut_sigs/ class=md-nav__link> Mutational Signatures </a> </li> <li class=md-nav__item> <a href=../../methods/S14_mut_timing/ class=md-nav__link> Molecular Timing of Somatic Drivers </a> </li> <li class=md-nav__item> <a href=../../methods/S15_class_predict_methy/ class=md-nav__link> Class Prediction using methylation classifier </a> </li> <li class=md-nav__item> <a href=../../methods/S16_expression_immune/ class=md-nav__link> Immune Microenvironment Analysis </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3 type=checkbox id=__nav_3> <label class=md-nav__link for=__nav_3> Figures <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Figures data-md-level=1> <label class=md-nav__title for=__nav_3> <span class="md-nav__icon md-icon"></span> Figures </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/A1_preload/ class=md-nav__link> Start Here </a> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_2 type=checkbox id=__nav_3_2> <label class=md-nav__link for=__nav_3_2> F1 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F1 data-md-level=2> <label class=md-nav__title for=__nav_3_2> <span class="md-nav__icon md-icon"></span> F1 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/F1A/ class=md-nav__link> F1A &middot; Somatic Mutational Landscape </a> </li> <li class=md-nav__item> <a href=../../figures/F1B/ class=md-nav__link> F1B &middot; Somatic Recurrent Hotspot Mutations </a> </li> <li class=md-nav__item> <a href=../../figures/F1C/ class=md-nav__link> F1C &middot; Comparative Pathway Analysis </a> </li> <li class=md-nav__item> <a href=../../figures/F1D/ class=md-nav__link> F1D &middot; Comparative Somatic Mutational Burden </a> </li> <li class=md-nav__item> <a href=../../figures/S1/ class=md-nav__link> S1A &middot; DNA fingerprinting </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_3 type=checkbox id=__nav_3_3> <label class=md-nav__link for=__nav_3_3> F2 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F2 data-md-level=2> <label class=md-nav__title for=__nav_3_3> <span class="md-nav__icon md-icon"></span> F2 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/F2A/ class=md-nav__link> F2A &middot; Somatic Copy Number Landscape </a> </li> <li class=md-nav__item> <a href=../../figures/F2B/ class=md-nav__link> F2B &middot; Comparative Aneuploidy Burden </a> </li> <li class=md-nav__item> <a href=../../figures/F2C/ class=md-nav__link> F2C &middot; Comparative Shared Syntenic Aneuploidy </a> </li> <li class=md-nav__item> <a href=../../figures/F2D/ class=md-nav__link> F2D &middot; Comparative Intratumor Heterogeneity </a> </li> <li class=md-nav__item> <a href=../../figures/S2/ class=md-nav__link> S2 &middot; IGV Plots </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_4 type=checkbox id=__nav_3_4> <label class=md-nav__link for=__nav_3_4> F3 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F3 data-md-level=2> <label class=md-nav__title for=__nav_3_4> <span class="md-nav__icon md-icon"></span> F3 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/F3A/ class=md-nav__link> F3A &middot; Somatic Mutational Signatures - Deconvoluted </a> </li> <li class=md-nav__item> <a href=../../figures/F3B/ class=md-nav__link> F3B &middot; Somatic Mutational Landscape - de novo </a> </li> <li class=md-nav__item> <a href=../../figures/F3C/ class=md-nav__link> F3C &middot; Comparative Somatic Mutational Signatures in Driver Genes </a> </li> <li class=md-nav__item> <a href=../../figures/F3D/ class=md-nav__link> F3D &middot; Comparative Molecular Timing of Driver Events </a> </li> <li class=md-nav__item> <a href=../../figures/S3/ class=md-nav__link> S3 &middot; Suppl Figures </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_5 type=checkbox id=__nav_3_5> <label class=md-nav__link for=__nav_3_5> F4 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F4 data-md-level=2> <label class=md-nav__title for=__nav_3_5> <span class="md-nav__icon md-icon"></span> F4 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/F4/ class=md-nav__link> F4 &middot; DNA Methylation based Prediction of Canine Gliomas </a> </li> <li class=md-nav__item> <a href=../../figures/S4/ class=md-nav__link> S4 &middot; Suppl Figure 4 </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_3_6 type=checkbox id=__nav_3_6> <label class=md-nav__link for=__nav_3_6> F5 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=F5 data-md-level=2> <label class=md-nav__title for=__nav_3_6> <span class="md-nav__icon md-icon"></span> F5 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../figures/F5/ class=md-nav__link> F5B &middot; Comparative Immunohistochemistry - Quantitative </a> </li> <li class=md-nav__item> <a href=../../figures/S5/ class=md-nav__link> S5 &middot; Suppl Figure 5 </a> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--active md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=__nav_4 type=checkbox id=__nav_4 checked> <div class="md-nav__link md-nav__link--index "> <a href=../ >R Markdown</a> <label for=__nav_4> <span class="md-nav__icon md-icon"></span> </label> </div> <nav class=md-nav aria-label="R Markdown" data-md-level=1> <label class=md-nav__title for=__nav_4> <span class="md-nav__icon md-icon"></span> R Markdown </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" data-md-toggle=toc type=checkbox id=__toc> <label class="md-nav__link md-nav__link--active" for=__toc> Output <span class="md-nav__icon md-icon"></span> </label> <a href=./ class="md-nav__link md-nav__link--active"> Output </a> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#comparative-molecular-life-history-of-spontaneous-canine-and-human-gliomas class=md-nav__link> Comparative molecular life history of spontaneous canine and human gliomas </a> </li> <li class=md-nav__item> <a href=#set-env class=md-nav__link> Set Env </a> <nav class=md-nav aria-label="Set Env"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#load-r-packages class=md-nav__link> Load R packages </a> </li> <li class=md-nav__item> <a href=#load-r-objects class=md-nav__link> Load R objects </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1a class=md-nav__link> Fig 1A </a> <nav class=md-nav aria-label="Fig 1A"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#plot-fig-1a class=md-nav__link> Plot Fig 1A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1b class=md-nav__link> Suppl Fig 1B </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1f class=md-nav__link> Suppl Fig 1F </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1b class=md-nav__link> Fig 1B </a> </li> <li class=md-nav__item> <a href=#fig-1c class=md-nav__link> Fig 1C </a> <nav class=md-nav aria-label="Fig 1C"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#fig-1c_1 class=md-nav__link> Fig 1C </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1c class=md-nav__link> Suppl Fig 1C </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1d class=md-nav__link> Fig 1D </a> <nav class=md-nav aria-label="Fig 1D"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-1d class=md-nav__link> Suppl Fig 1D </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1e class=md-nav__link> Suppl Fig 1E </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#f2a class=md-nav__link> F2A </a> </li> <li class=md-nav__item> <a href=#fig-2b class=md-nav__link> Fig 2B </a> <nav class=md-nav aria-label="Fig 2B"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-2e class=md-nav__link> Suppl Fig 2E </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-2c class=md-nav__link> Fig 2C </a> </li> <li class=md-nav__item> <a href=#fig-2d class=md-nav__link> Fig 2D </a> <nav class=md-nav aria-label="Fig 2D"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-2d class=md-nav__link> Suppl Fig 2D </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3a class=md-nav__link> Fig 3A </a> <nav class=md-nav aria-label="Fig 3A"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_1 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-3a_1 class=md-nav__link> Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3a class=md-nav__link> Suppl Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3b class=md-nav__link> Suppl Fig 3B </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3b class=md-nav__link> Fig 3B </a> </li> <li class=md-nav__item> <a href=#fig-3c class=md-nav__link> Fig 3C </a> <nav class=md-nav aria-label="Fig 3C"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#winning-tables class=md-nav__link> Winning Tables </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3d class=md-nav__link> Fig 3D </a> </li> <li class=md-nav__item> <a href=#fig-4 class=md-nav__link> Fig 4 </a> <nav class=md-nav aria-label="Fig 4"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#preprocessing class=md-nav__link> Preprocessing </a> <nav class=md-nav aria-label=Preprocessing> <ul class=md-nav__list> <li class=md-nav__item> <a href=#format-data-for-plotting class=md-nav__link> Format data for plotting </a> </li> <li class=md-nav__item> <a href=#create-table-for-plotting-annotations class=md-nav__link> Create table for plotting annotations </a> </li> <li class=md-nav__item> <a href=#labelscolors-for-plot class=md-nav__link> Labels/colors for plot </a> </li> <li class=md-nav__item> <a href=#plot-probabilities-and-predictions-data-frame-simultaneously class=md-nav__link> Plot probabilities and predictions data frame simultaneously </a> </li> <li class=md-nav__item> <a href=#add-annotations-for-mutation-rate-purity-and-age class=md-nav__link> Add annotations for mutation rate, purity, and age </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-4_1 class=md-nav__link> Fig 4 </a> </li> <li class=md-nav__item> <a href=#suppl-fig-4a class=md-nav__link> Suppl Fig 4A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-4b class=md-nav__link> Suppl Fig 4B </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-5 class=md-nav__link> Fig 5 </a> <nav class=md-nav aria-label="Fig 5"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_2 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-5b class=md-nav__link> Fig 5B </a> </li> <li class=md-nav__item> <a href=#p-values class=md-nav__link> p-values </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-5_1 class=md-nav__link> Fig 5 </a> <nav class=md-nav aria-label="Fig 5"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_3 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-s5 class=md-nav__link> Fig S5 </a> </li> </ul> </nav> </li> </ul> </nav> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=https://verhaaklab.com/contact class=md-nav__link> Contact </a> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=sidebar data-md-type=toc> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label="Table of contents"> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> Table of contents </label> <ul class=md-nav__list data-md-component=toc data-md-scrollfix> <li class=md-nav__item> <a href=#comparative-molecular-life-history-of-spontaneous-canine-and-human-gliomas class=md-nav__link> Comparative molecular life history of spontaneous canine and human gliomas </a> </li> <li class=md-nav__item> <a href=#set-env class=md-nav__link> Set Env </a> <nav class=md-nav aria-label="Set Env"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#load-r-packages class=md-nav__link> Load R packages </a> </li> <li class=md-nav__item> <a href=#load-r-objects class=md-nav__link> Load R objects </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1a class=md-nav__link> Fig 1A </a> <nav class=md-nav aria-label="Fig 1A"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#plot-fig-1a class=md-nav__link> Plot Fig 1A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1b class=md-nav__link> Suppl Fig 1B </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1f class=md-nav__link> Suppl Fig 1F </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1b class=md-nav__link> Fig 1B </a> </li> <li class=md-nav__item> <a href=#fig-1c class=md-nav__link> Fig 1C </a> <nav class=md-nav aria-label="Fig 1C"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#fig-1c_1 class=md-nav__link> Fig 1C </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1c class=md-nav__link> Suppl Fig 1C </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-1d class=md-nav__link> Fig 1D </a> <nav class=md-nav aria-label="Fig 1D"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-1d class=md-nav__link> Suppl Fig 1D </a> </li> <li class=md-nav__item> <a href=#suppl-fig-1e class=md-nav__link> Suppl Fig 1E </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#f2a class=md-nav__link> F2A </a> </li> <li class=md-nav__item> <a href=#fig-2b class=md-nav__link> Fig 2B </a> <nav class=md-nav aria-label="Fig 2B"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-2e class=md-nav__link> Suppl Fig 2E </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-2c class=md-nav__link> Fig 2C </a> </li> <li class=md-nav__item> <a href=#fig-2d class=md-nav__link> Fig 2D </a> <nav class=md-nav aria-label="Fig 2D"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#suppl-fig-2d class=md-nav__link> Suppl Fig 2D </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3a class=md-nav__link> Fig 3A </a> <nav class=md-nav aria-label="Fig 3A"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_1 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-3a_1 class=md-nav__link> Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3a class=md-nav__link> Suppl Fig 3A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-3b class=md-nav__link> Suppl Fig 3B </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3b class=md-nav__link> Fig 3B </a> </li> <li class=md-nav__item> <a href=#fig-3c class=md-nav__link> Fig 3C </a> <nav class=md-nav aria-label="Fig 3C"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#winning-tables class=md-nav__link> Winning Tables </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-3d class=md-nav__link> Fig 3D </a> </li> <li class=md-nav__item> <a href=#fig-4 class=md-nav__link> Fig 4 </a> <nav class=md-nav aria-label="Fig 4"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#preprocessing class=md-nav__link> Preprocessing </a> <nav class=md-nav aria-label=Preprocessing> <ul class=md-nav__list> <li class=md-nav__item> <a href=#format-data-for-plotting class=md-nav__link> Format data for plotting </a> </li> <li class=md-nav__item> <a href=#create-table-for-plotting-annotations class=md-nav__link> Create table for plotting annotations </a> </li> <li class=md-nav__item> <a href=#labelscolors-for-plot class=md-nav__link> Labels/colors for plot </a> </li> <li class=md-nav__item> <a href=#plot-probabilities-and-predictions-data-frame-simultaneously class=md-nav__link> Plot probabilities and predictions data frame simultaneously </a> </li> <li class=md-nav__item> <a href=#add-annotations-for-mutation-rate-purity-and-age class=md-nav__link> Add annotations for mutation rate, purity, and age </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-4_1 class=md-nav__link> Fig 4 </a> </li> <li class=md-nav__item> <a href=#suppl-fig-4a class=md-nav__link> Suppl Fig 4A </a> </li> <li class=md-nav__item> <a href=#suppl-fig-4b class=md-nav__link> Suppl Fig 4B </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-5 class=md-nav__link> Fig 5 </a> <nav class=md-nav aria-label="Fig 5"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_2 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-5b class=md-nav__link> Fig 5B </a> </li> <li class=md-nav__item> <a href=#p-values class=md-nav__link> p-values </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#fig-5_1 class=md-nav__link> Fig 5 </a> <nav class=md-nav aria-label="Fig 5"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#required-metadata_3 class=md-nav__link> Required metadata </a> </li> <li class=md-nav__item> <a href=#fig-s5 class=md-nav__link> Fig S5 </a> </li> </ul> </nav> </li> </ul> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <!-- Add copied contents from original content.html here --> <!-- https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/src/partials/content.html --> <!-- Edit button --> <!--
  Hack: check whether the content contains a h1 headline. If it
  doesn't, the page title (or respectively site name) is used
  as the main headline.
--> <h1>Output</h1> <!-- Markdown content --> <h2 id=comparative-molecular-life-history-of-spontaneous-canine-and-human-gliomas>Comparative molecular life history of spontaneous canine and human gliomas<a class=headerlink href=#comparative-molecular-life-history-of-spontaneous-canine-and-human-gliomas title="Permanent link">&para;</a></h2> <blockquote> <p>Amin et al., Comparative Molecular Life History of Spontaneous Canine and Human Gliomas, <em>Cancer Cell</em> (2020). DOI: <a href=https://doi.org/10.1016/j.ccell.2020.01.004>10.1016/j.ccell.2020.01.004</a> Feb 10, 2020.</p> </blockquote> <h2 id=set-env>Set Env<a class=headerlink href=#set-env title="Permanent link">&para;</a></h2> <div class=highlight><pre><span></span><code><span class=nf>library</span><span class=p>(</span><span class=n>knitr</span><span class=p>)</span>
<span class=n>opts_chunk</span><span class=o>$</span><span class=nf>set</span><span class=p>(</span><span class=n>comment</span> <span class=o>=</span> <span class=kc>NA</span><span class=p>,</span> <span class=n>fig.width</span> <span class=o>=</span> <span class=m>18</span><span class=p>,</span> <span class=n>fig.height</span> <span class=o>=</span> <span class=m>12</span><span class=p>,</span> 
               <span class=n>message</span> <span class=o>=</span> <span class=kc>TRUE</span><span class=p>,</span> <span class=n>cache</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>,</span> <span class=n>echo</span> <span class=o>=</span> <span class=kc>FALSE</span><span class=p>)</span>
</code></pre></div> <h3 id=load-r-packages>Load R packages<a class=headerlink href=#load-r-packages title="Permanent link">&para;</a></h3> <div class=highlight><pre><span></span><code>Warning: replacing previous import &#39;NMF::entropy&#39; by &#39;lsa::entropy&#39; when loading
&#39;Palimpsest&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>Warning: replacing previous import &#39;NMF::dispersion&#39; by &#39;plotrix::dispersion&#39;
when loading &#39;Palimpsest&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>Warning: replacing previous import &#39;gplots::plotCI&#39; by &#39;plotrix::plotCI&#39; when
loading &#39;Palimpsest&#39;
</code></pre></div> <h3 id=load-r-objects>Load R objects<a class=headerlink href=#load-r-objects title="Permanent link">&para;</a></h3> <div class=highlight><pre><span></span><code>Work directory is /Users/samin1/Dropbox (Personal)/acad/scripts/HOURGLASS/cgp_keystone/canineglioma/docs/run_code
Loading base objects...
</code></pre></div> <div class=highlight><pre><span></span><code>List objects
</code></pre></div> <div class=highlight><pre><span></span><code> [1] &quot;ALL_HALLMARK_v1&quot;                     
 [2] &quot;cancer_genes_maftools&quot;               
 [3] &quot;cancer_signatures&quot;                   
 [4] &quot;CG_all&quot;                              
 [5] &quot;cgp_aneuploidy_metrics_master&quot;       
 [6] &quot;cgp_cibersort&quot;                       
 [7] &quot;cgp_info&quot;                            
 [8] &quot;cgp_maftools_gistic_matched&quot;         
 [9] &quot;cgp_maftools_gistic_methy&quot;           
[10] &quot;cgp_maftools_gistic_n81&quot;             
[11] &quot;cgp_RRBS.noNormals_class_prediction&quot; 
[12] &quot;cgp_RRBS.noNormals_sample_info&quot;      
[13] &quot;cgp_suppl_sample_info_master&quot;        
[14] &quot;driver_sigs_groups&quot;                  
[15] &quot;fig5b_dat&quot;                           
[16] &quot;fit_res_hm&quot;                          
[17] &quot;fit_res_nonhm&quot;                       
[18] &quot;HALLMARK&quot;                            
[19] &quot;hm_nonhm_topsigs_tbl&quot;                
[20] &quot;instability_sigs_df&quot;                 
[21] &quot;maf_nogistic&quot;                        
[22] &quot;mat_aneuploidy_summary_mostvar_cgAll&quot;
[23] &quot;merged_ag_pg_cg_denovo_sig&quot;          
[24] &quot;merged_anp_scores&quot;                   
[25] &quot;merged_ccf_vaf_df&quot;                   
[26] &quot;mut_mat&quot;                             
[27] &quot;mutrate_table&quot;                       
[28] &quot;ped_cibersort&quot;                       
[29] &quot;tcga_cibersort&quot;                      
[30] &quot;tcga_info&quot;                           
[31] &quot;trimmed_ag_vcf&quot;                      
[32] &quot;trimmed_cg_vcf&quot;                      
[33] &quot;trimmed_pg_vcf&quot;                      
[34] &quot;updated_merged_entropy_df_ridges&quot;    
[35] &quot;win_table_ag_vcf&quot;                    
[36] &quot;win_table_cg_vcf&quot;                    
[37] &quot;win_table_pg_vcf&quot;                    
</code></pre></div> <hr> <h2 id=fig-1a>Fig 1A<a class=headerlink href=#fig-1a title="Permanent link">&para;</a></h2> <h3 id=required-metadata>Required metadata<a class=headerlink href=#required-metadata title="Permanent link">&para;</a></h3> <ul> <li>Genes to plot</li> </ul> <blockquote> <p>Based on <a href=/methods/S06_smgs>significantly mutated genes analysis</a>. See also Table S2.</p> </blockquote> <ul> <li>Sample and Variant Annotations </li> </ul> <h3 id=plot-fig-1a>Plot Fig 1A<a class=headerlink href=#plot-fig-1a title="Permanent link">&para;</a></h3> <blockquote> <p>Somatic mutational lanscape of glioma driver genes. Based on <a href=/methods/S06_smgs>significantly mutated genes analysis</a>.</p> </blockquote> <details class=error><summary>font family 'Arial' not found</summary><p>If you get an error, <code>font family 'Arial' not found in PostScript font database</code>, either remove <em>fonts</em> argument from <code>pdf</code> or import system fonts using <em>extrafont</em> R package. See <a href=https://www.fromthebottomoftheheap.net/2013/09/09/preparing-figures-for-plos-one-with-r/ >documentation by Gavin Simpson</a> for details. Run <code>extrafont::loadfonts()</code> as detailed in <a href=/figures/a1_preload/#load-r-packages>Start Here</a>.</p> </details> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=suppl-fig-1b>Suppl Fig 1B<a class=headerlink href=#suppl-fig-1b title="Permanent link">&para;</a></h3> <blockquote> <p>Somatic mutational lanscape of COSMIC cancer genes.</p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=suppl-fig-1f>Suppl Fig 1F<a class=headerlink href=#suppl-fig-1f title="Permanent link">&para;</a></h3> <blockquote> <p>Somatic mutational lanscape, including copy-number aberrations of COSMIC cancer genes. This figure also contains somatic copy-number data which is detailed under <a href=/figures/F2A/ >Fig 2A</a>.</p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <hr> <h2 id=fig-1b>Fig 1B<a class=headerlink href=#fig-1b title="Permanent link">&para;</a></h2> <ul> <li>Alternate code using <code>lollipopPlot</code> function in R package, <a href=https://bioconductor.org/packages/release/bioc/html/maftools.html><em>maftools</em></a></li> </ul> <div class=highlight><pre><span></span><code>    pos   conv count Variant_Classification
1:   88   R88Q     1      Missense_Mutation
2:  542  E542K     1      Missense_Mutation
3:  545  E545K     1      Missense_Mutation
4: 1047 H1047R     7      Missense_Mutation
5: 1047 H1047L     1      Missense_Mutation
</code></pre></div> <div class=highlight><pre><span></span><code>   pos  conv count Variant_Classification
1: 132 R132C     3      Missense_Mutation
</code></pre></div> <div class=highlight><pre><span></span><code>6 transcripts available. Use arguments refSeqID or proteinID to manually specify tx name.
</code></pre></div> <div class=highlight><pre><span></span><code>   HGNC    refseq.ID   protein.ID aa.length
1: SPOP NM_001007227 NP_001007228       374
2: SPOP NM_001007228 NP_001007229       374
3: SPOP NM_001007229 NP_001007230       374
4: SPOP NM_001007230 NP_001007231       374
5: SPOP NM_001007226 NP_001007227       374
6: SPOP    NM_003563    NP_003554       374
</code></pre></div> <div class=highlight><pre><span></span><code>Using longer transcript NM_001007227 for now.
</code></pre></div> <div class=highlight><pre><span></span><code>   pos conv count Variant_Classification
1:  94 P94R     2      Missense_Mutation
</code></pre></div> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <hr> <h2 id=fig-1c>Fig 1C<a class=headerlink href=#fig-1c title="Permanent link">&para;</a></h2> <p>Cancer Hallmarks enrichment across canine, human pediatric and adult gliomas. </p> <blockquote> <p>Genes to hallmark mapping was primarily derived from SLAPenrich R package. See also <a href=/methods/S07_hallmarks/ >relevant methods</a> for details.</p> </blockquote> <div class=highlight><pre><span></span><code>Joining, by = &quot;hallmark&quot;
</code></pre></div> <div class=highlight><pre><span></span><code>Warning: Column `hallmark` joining character vector and factor, coercing into
character vector
</code></pre></div> <ul> <li>Similar to <code>CG_HALLMARK</code>, enrichment per hallmark tables were created for molecular subtypes across human cancers, and merged in the master table, <code>ALL_HALLMARK_v1</code>.</li> </ul> <h3 id=fig-1c_1>Fig 1C<a class=headerlink href=#fig-1c_1 title="Permanent link">&para;</a></h3> <blockquote> <p>Subset of hallmarks as a main figure 1C.</p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=suppl-fig-1c>Suppl Fig 1C<a class=headerlink href=#suppl-fig-1c title="Permanent link">&para;</a></h3> <blockquote> <p>Visualize all hallmarks.</p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <hr> <h2 id=fig-1d>Fig 1D<a class=headerlink href=#fig-1d title="Permanent link">&para;</a></h2> <blockquote> <p>Plot CNS tumors.</p> </blockquote> <h3 id=suppl-fig-1d>Suppl Fig 1D<a class=headerlink href=#suppl-fig-1d title="Permanent link">&para;</a></h3> <blockquote> <p>Plot pancancer tumors</p> </blockquote> <details class=error><summary>unexpected <code>,</code> error</summary><p>Running code by copy-paste may result in error related to R syntax, e.g., <code>Error: unexpected ',' in "</code>. If so, try to copy code first in text editor or RStudio app with lint feature. Please <a href=https://github.com/TheJacksonLaboratory/canineglioma/issues>report bugs</a> by submitting a GitHub issue.</p> </details> <h3 id=suppl-fig-1e>Suppl Fig 1E<a class=headerlink href=#suppl-fig-1e title="Permanent link">&para;</a></h3> <blockquote> <p>Comparing mutational burden across molecular subtypes.</p> </blockquote> <div class=highlight><pre><span></span><code>Warning in if (fill %in% names(data) &amp; is.null(add.params$fill)) add.params$fill
&lt;- fill: the condition has length &gt; 1 and only the first element will be used
</code></pre></div> <hr> <h2 id=f2a>F2A<a class=headerlink href=#f2a title="Permanent link">&para;</a></h2> <blockquote> <p>Somatic copy-number aberrations (SCNA) lanscape of glioma driver genes. Please run required code from <a href=/figures/F1A/ >Fig 1A</a> prior to running code from this page.</p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <blockquote> <p>Somatic copy numbers were classified as follows based on GISTIC2 values.</p> </blockquote> <table> <thead> <tr> <th>GISTIC2 value</th> <th>Estimated copy number</th> <th>Classification</th> </tr> </thead> <tbody> <tr> <td><span class=arithmatex>\(&lt; -1.29\)</span></td> <td>-2</td> <td>Deep deletion</td> </tr> <tr> <td><span class=arithmatex>\(-1.29 &lt;&gt; -.65\)</span></td> <td>-1</td> <td>Loss of Heterozygosity (LOH)</td> </tr> <tr> <td><span class=arithmatex>\(-.65 &lt;&gt; 2.0\)</span></td> <td>0-1</td> <td>Copy-neutral or low-level amplification</td> </tr> <tr> <td><span class=arithmatex>\(&gt;2.0\)</span></td> <td>2</td> <td>High-level amplification</td> </tr> </tbody> </table> <hr> <h2 id=fig-2b>Fig 2B<a class=headerlink href=#fig-2b title="Permanent link">&para;</a></h2> <blockquote> <p>Comparative aneuploidy burden. Related code is available at the GLASS consortium <a href=https://github.com/TheJacksonLaboratory/GLASS/tree/master/sql/cnv>marker paper code repository</a>, namely, <code>recapseg_postgres.sql</code> and <code>taylor_aneuploidy.sql</code>.</p> </blockquote> <div class=highlight><pre><span></span><code>Warning in if (fill %in% names(data) &amp; is.null(add.params$fill)) add.params$fill
&lt;- fill: the condition has length &gt; 1 and only the first element will be used
</code></pre></div> <div class=highlight><pre><span></span><code>notch went outside hinges. Try setting notch=FALSE.
notch went outside hinges. Try setting notch=FALSE.
</code></pre></div> <h3 id=suppl-fig-2e>Suppl Fig 2E<a class=headerlink href=#suppl-fig-2e title="Permanent link">&para;</a></h3> <blockquote> <p>Aneuploidy vs. somatic mutation burden.</p> </blockquote> <hr> <h2 id=fig-2c>Fig 2C<a class=headerlink href=#fig-2c title="Permanent link">&para;</a></h2> <p><code>mat_aneuploidy_summary_mostvar_cgAll</code> matrix contains proportion of patients per cohort with arm-level aneuploidy, and was based on <a href=/methods/S11_aneuploidy/ >arm-level aneuploidy metrics</a>. Related code is available at the GLASS consortium <a href=https://github.com/TheJacksonLaboratory/GLASS/tree/master/sql/cnv>marker paper code repository</a>, namely, <code>recapseg_postgres.sql</code> and <code>taylor_aneuploidy.sql</code>.</p> <hr> <h2 id=fig-2d>Fig 2D<a class=headerlink href=#fig-2d title="Permanent link">&para;</a></h2> <blockquote> <p>Clonality of potential driver genes with respect to intra-tumor heterogeneity as measured with Shannon entropy.</p> </blockquote> <details class=error><summary>Error: unexpected symbol in...<code>who_subtype</code>...</summary><p>Prefer runnning code in RStudio. This error does not appear when running code in RStudio but pops up when running on command-line R. Likely due to font ligatures and/or use of of line breaks.</p> </details> <h3 id=suppl-fig-2d>Suppl Fig 2D<a class=headerlink href=#suppl-fig-2d title="Permanent link">&para;</a></h3> <blockquote> <p>Comparative intra-tumor heterogeneity measured using Shannon entropy.</p> </blockquote> <div class=highlight><pre><span></span><code>Warning in if (fill %in% names(data) &amp; is.null(add.params$fill)) add.params$fill
&lt;- fill: the condition has length &gt; 1 and only the first element will be used
</code></pre></div> <hr> <h2 id=fig-3a>Fig 3A<a class=headerlink href=#fig-3a title="Permanent link">&para;</a></h2> <h3 id=required-metadata_1>Required metadata<a class=headerlink href=#required-metadata_1 title="Permanent link">&para;</a></h3> <ul> <li>Canine glioma patients with <a href=/methods/S13_mut_sigs/ >outlier mutation profile</a>.</li> </ul> <div class=highlight><pre><span></span><code>Call:
lm(formula = adusted_coding_mutrate ~ Matched_Normal + Tissue_Archival, 
    data = cgp_sample_info_maftools_hypermuts)

Residuals:
    Min      1Q  Median      3Q     Max 
-1.4620 -0.2753 -0.1240  0.1106  6.0669 

Coefficients:
                           Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)                 0.36934    0.14806   2.495   0.0148 *  
Matched_Normaltumor_only    1.49685    0.24842   6.025 5.62e-08 ***
Tissue_Archivalsnap-frozen  0.05463    0.19010   0.287   0.7746    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.8392 on 76 degrees of freedom
Multiple R-squared:  0.3238,    Adjusted R-squared:  0.306 
F-statistic:  18.2 on 2 and 76 DF,  p-value: 3.492e-07
</code></pre></div> <div class=highlight><pre><span></span><code>[1] &quot;hypermut case: i_8228&quot; &quot;hypermut case: i_95FC&quot;
</code></pre></div> <div class=highlight><pre><span></span><code>[1] &quot;outlier among non_hypermut: i_BD58&quot; &quot;outlier among non_hypermut: i_1C36&quot;
[3] &quot;outlier among non_hypermut: i_502F&quot; &quot;outlier among non_hypermut: i_D732&quot;
[5] &quot;outlier among non_hypermut: i_6B06&quot; &quot;outlier among non_hypermut: i_56B5&quot;
</code></pre></div> <div class=highlight><pre><span></span><code>[1] &quot;i_8228&quot; &quot;i_95FC&quot; &quot;i_BD58&quot; &quot;i_1C36&quot; &quot;i_502F&quot; &quot;i_D732&quot; &quot;i_6B06&quot; &quot;i_56B5&quot;
</code></pre></div> <blockquote> <p>"i_8228" "i_95FC" "i_BD58" "i_1C36" "i_502F" "i_D732" "i_6B06" "i_56B5"</p> </blockquote> <ul> <li> <p>Pick random 9/73 cases without outlier mutation profile.</p> </li> <li> <p>Deconvolute known human mutational signatures</p> </li> </ul> <blockquote> <p>Fitted mutational signatures, i.e., <code>fit_res</code>, <code>fit_res_hm</code> and <code>fit_res_nonhm</code> were derived using <code>fit_to_signatures</code> function from <em>MutationalPatterns</em> R package.</p> </blockquote> <ul> <li>Merge signatures by proposed mechanism</li> </ul> <div class=highlight><pre><span></span><code>Joining, by = &quot;sig_id&quot;
Joining, by = &quot;sig_id&quot;
Joining, by = &quot;sig_id&quot;
</code></pre></div> <ul> <li>Function to plot signature contribution</li> </ul> <blockquote> <p>Rewrite of <code>plot_contribution</code> function from <em>MutationalPatterns</em> R package.</p> </blockquote> <h3 id=fig-3a_1>Fig 3A<a class=headerlink href=#fig-3a_1 title="Permanent link">&para;</a></h3> <blockquote> <p>Plot deconvoluted signature contribution after grouping signatures based on proposed mechanism.</p> </blockquote> <h3 id=suppl-fig-3a>Suppl Fig 3A<a class=headerlink href=#suppl-fig-3a title="Permanent link">&para;</a></h3> <blockquote> <p>Plot absolute and relative contribution of dominant signatures.</p> </blockquote> <div class=highlight><pre><span></span><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  0.000   0.000   0.000   3.282   2.750  35.465 
</code></pre></div> <div class=highlight><pre><span></span><code> [1] &quot;Signature 1&quot;    &quot;Signature 2&quot;    &quot;Signature 8&quot;    &quot;Signature 9&quot;   
 [5] &quot;Signature 11&quot;   &quot;Signature 15&quot;   &quot;Signature 27&quot;   &quot;Signature T-1&quot; 
 [9] &quot;Signature T-3&quot;  &quot;Signature T-10&quot; &quot;Signature T-11&quot;
</code></pre></div> <h3 id=suppl-fig-3b>Suppl Fig 3B<a class=headerlink href=#suppl-fig-3b title="Permanent link">&para;</a></h3> <blockquote> <p>Boxplot comparison of genomic instability related signatures between canine glioma patients separated by outlier mutation profile.</p> </blockquote> <div class=highlight><pre><span></span><code>notch went outside hinges. Try setting notch=FALSE.
notch went outside hinges. Try setting notch=FALSE.
notch went outside hinges. Try setting notch=FALSE.
notch went outside hinges. Try setting notch=FALSE.
notch went outside hinges. Try setting notch=FALSE.
</code></pre></div> <hr> <h2 id=fig-3b>Fig 3B<a class=headerlink href=#fig-3b title="Permanent link">&para;</a></h2> <blockquote> <p>Compare de-novo mutational signatures in canine and human pediatric and adult glioma with deconvoluted ones from Human Cancers (n=42).</p> </blockquote> <ul> <li> <p>For each of three cohorts (canine, human pediatric and adult glioma), de-novo signatures were constructed as follows:</p> </li> <li> <p>Resulting de-novo signatures from three cohorts were then merged into a single dataframe.</p> </li> <li> <p>Calculate cosine similarity using <code>deconvolution_compare</code> function in <a href=https://github.com/FunGeST/Palimpsest><em>Palimpsest</em></a> R package. Author of Palimpsest package has provided <a href=https://github.com/FunGeST/Palimpsest/raw/master/Files/vignette_palimpsest_2.0.pdf>a detailed documentation</a> on how to run Palimpsest workflow.</p> </li> </ul> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <hr> <h2 id=fig-3c>Fig 3C<a class=headerlink href=#fig-3c title="Permanent link">&para;</a></h2> <blockquote> <p>Plot estimated mutational signature contribution per driver gene.</p> </blockquote> <div class=highlight><pre><span></span><code>Joining, by = &quot;sigs&quot;
Joining, by = &quot;sigs&quot;
Joining, by = &quot;sigs&quot;
</code></pre></div> <hr> <h3 id=winning-tables>Winning Tables<a class=headerlink href=#winning-tables title="Permanent link">&para;</a></h3> <blockquote> <p>Make winning matrix of somatic variants in cancer driver genes (n=73) based on inferred clonality using <a href=https://github.com/FunGeST/Palimpsest><em>Palimpsest</em></a> R package. Author of Palimpsest package has provided <a href=https://github.com/FunGeST/Palimpsest/raw/master/Files/vignette_palimpsest_2.0.pdf>a detailed documentation</a> on how to run Palimpsest workflow.</p> </blockquote> <h2 id=fig-3d>Fig 3D<a class=headerlink href=#fig-3d title="Permanent link">&para;</a></h2> <blockquote> <p>Run <em>Bradley-Terry model</em> to get <a href=/methods/S14_mut_timing/ >estimated relative timing of somatic driver events</a>.</p> </blockquote> <div class=highlight><pre><span></span><code>Attaching package: &#39;igraph&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:rtracklayer&#39;:

    blocks, path
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:GenomicRanges&#39;:

    union
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:IRanges&#39;:

    union
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:S4Vectors&#39;:

    union
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:NMF&#39;:

    algorithm, compare
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:BiocGenerics&#39;:

    normalize, path, union
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:dplyr&#39;:

    as_data_frame, groups, union
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:purrr&#39;:

    compose, simplify
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:tidyr&#39;:

    crossing
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:tibble&#39;:

    as_data_frame
</code></pre></div> <div class=highlight><pre><span></span><code>The following objects are masked from &#39;package:stats&#39;:

    decompose, spectrum
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:base&#39;:

    union
</code></pre></div> <div class=highlight><pre><span></span><code>Attaching package: &#39;ggridges&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:ggplot2&#39;:

    scale_discrete_manual
</code></pre></div> <div class=highlight><pre><span></span><code>Number of items: 36 
Density of wins matrix: 0.2114198 
Fully-connected: FALSE 
Number of fully-connected components: 15 
Summary of fully-connected components: 
  Component size Freq
1              1   10
2              2    2
3              4    1
4              9    2
  Component size Freq
1              1   10
2              2    2
3              4    1
4              9    2
</code></pre></div> <div class=highlight><pre><span></span><code>Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead

Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead
</code></pre></div> <div class=highlight><pre><span></span><code>Number of items: 53 
Density of wins matrix: 0.1893912 
Fully-connected: FALSE 
Number of fully-connected components: 35 
Summary of fully-connected components: 
  Component size Freq
1              1   30
2              2    1
3              3    3
4             12    1
  Component size Freq
1              1   30
2              2    1
3              3    3
4             12    1
</code></pre></div> <div class=highlight><pre><span></span><code>Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead

Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead
</code></pre></div> <div class=highlight><pre><span></span><code>Number of items: 41 
Density of wins matrix: 0.2064247 
Fully-connected: FALSE 
Number of fully-connected components: 18 
Summary of fully-connected components: 
  Component size Freq
1              1   13
2              2    2
3              4    1
4              7    1
5             13    1
  Component size Freq
1              1   13
2              2    2
3              4    1
4              7    1
5             13    1
</code></pre></div> <div class=highlight><pre><span></span><code>Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead

Warning in ref_check(ref, pi): The value of &#39;ref&#39; is not an item name. Using ref
= NULL instead
</code></pre></div> <div class=highlight><pre><span></span><code>Picking joint bandwidth of 0.0925
</code></pre></div> <div class=highlight><pre><span></span><code>Picking joint bandwidth of 0.118
</code></pre></div> <div class=highlight><pre><span></span><code>Picking joint bandwidth of 0.057
</code></pre></div> <hr> <h2 id=fig-4>Fig 4<a class=headerlink href=#fig-4 title="Permanent link">&para;</a></h2> <blockquote> <p>DNA methylation based classification of canine glioma.</p> </blockquote> <h3 id=preprocessing>Preprocessing<a class=headerlink href=#preprocessing title="Permanent link">&para;</a></h3> <h4 id=format-data-for-plotting>Format data for plotting<a class=headerlink href=#format-data-for-plotting title="Permanent link">&para;</a></h4> <h4 id=create-table-for-plotting-annotations>Create table for plotting annotations<a class=headerlink href=#create-table-for-plotting-annotations title="Permanent link">&para;</a></h4> <h4 id=labelscolors-for-plot>Labels/colors for plot<a class=headerlink href=#labelscolors-for-plot title="Permanent link">&para;</a></h4> <h4 id=plot-probabilities-and-predictions-data-frame-simultaneously>Plot probabilities and predictions data frame simultaneously<a class=headerlink href=#plot-probabilities-and-predictions-data-frame-simultaneously title="Permanent link">&para;</a></h4> <h4 id=add-annotations-for-mutation-rate-purity-and-age>Add annotations for mutation rate, purity, and age<a class=headerlink href=#add-annotations-for-mutation-rate-purity-and-age title="Permanent link">&para;</a></h4> <h3 id=fig-4_1>Fig 4<a class=headerlink href=#fig-4_1 title="Permanent link">&para;</a></h3> <div class=highlight><pre><span></span><code>Loading required package: magrittr
</code></pre></div> <div class=highlight><pre><span></span><code>Attaching package: &#39;magrittr&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:purrr&#39;:

    set_names
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:tidyr&#39;:

    extract
</code></pre></div> <div class=highlight><pre><span></span><code>Attaching package: &#39;ggpubr&#39;
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:egg&#39;:

    ggarrange
</code></pre></div> <div class=highlight><pre><span></span><code>The following object is masked from &#39;package:cowplot&#39;:

    get_legend
</code></pre></div> <div class=highlight><pre><span></span><code>Warning in as_grob.default(plot): Cannot convert object of class unit into a
grob.
</code></pre></div> <div class=highlight><pre><span></span><code>$`1`
</code></pre></div> <div class=highlight><pre><span></span><code>$`2`
</code></pre></div> <div class=highlight><pre><span></span><code>attr(,&quot;class&quot;)
[1] &quot;list&quot;      &quot;ggarrange&quot;
</code></pre></div> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=suppl-fig-4a>Suppl Fig 4A<a class=headerlink href=#suppl-fig-4a title="Permanent link">&para;</a></h3> <blockquote> <p>Somatic mutational lanscape, including copy-number aberrations of COSMIC cancer genes, grouped by <a href=/methods/S15_class_predict_methy/ >DNA methylation classification</a>. </p> </blockquote> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=suppl-fig-4b>Suppl Fig 4B<a class=headerlink href=#suppl-fig-4b title="Permanent link">&para;</a></h3> <blockquote> <p>DNA methylation age of human and canine glioma.</p> </blockquote> <p>Brewing...</p> <hr> <h2 id=fig-5>Fig 5<a class=headerlink href=#fig-5 title="Permanent link">&para;</a></h2> <blockquote> <p>IHC Panels</p> </blockquote> <h3 id=required-metadata_2>Required metadata<a class=headerlink href=#required-metadata_2 title="Permanent link">&para;</a></h3> <blockquote> <p>There were 33029 total fields measured and 27 fields with missing data. Fields with missing positivies were removed from further analysis.</p> </blockquote> <ul> <li>Number of fields by marker</li> </ul> <p>Marker # Fields</p> <hr> <p>CD14 7298 CD163 6601 CD3 6654 CD79A 5993 IBA1 6456</p> <ul> <li> <p>Counts of number of fields by marker for each sample:</p> <div class=highlight><pre><span></span><code>       Type         Grade       CD14    CD163    CD3     CD79A    IBA1
</code></pre></div> <hr> <p>A1 Adult Low Grade 123 155 189 142 145<br> A10 Adult High Grade 520 675 485 783 792<br> A11 Adult High Grade 240 188 221 276 234<br> A2 Adult Low Grade 422 288 300 245 300<br> A3 Adult High Grade 392 482 362 343 525<br> A4 Adult High Grade 746 689 595 473 623<br> A5 Adult High Grade 695 432 354 357 372<br> A6 Adult High Grade 271 232 130 185 248<br> A7 Adult High Grade 467 489 387 404 227<br> A8 Adult Low Grade 558 687 548 615 467<br> A9 Adult High Grade 220 192 166 194 212<br> N09-531 Canine High Grade 189 149 148 127 166<br> N09-547 Canine High Grade 68 65 57 78 58<br> N09-651 Canine High Grade 22 18 15 40 21<br> N09-868 Canine High Grade 85 47 31 39 56<br> N10-159 Canine Low Grade 55 54 46 33 57<br> N10-702 Canine Low Grade 105 47 38 21 50<br> N12-166 Canine Low Grade 223 125 153 148 163<br> N13-901 Canine High Grade 225 170 181 140 149<br> N15-161 Canine High Grade 77 52 113 64 117<br> N15-604 Canine High Grade 67 70 75 68 60<br> N16-199 Canine High Grade 43 29 41 48 29<br> 11434 Pediatric High Grade 42 33 70 38 84<br> KS18-1269 Pediatric High Grade 480 492 434 468 460<br> KS19-607 Pediatric Low Grade 54 66 48 40 0 <br> S13-28076 Pediatric High Grade 362 375 1222 334 559<br> S14-12021 Pediatric High Grade 547 300 245 290 282 </p> </li> <li> <p>Counts of different types of samples (ped, adult, or breed):</p> </li> </ul> <p>Sample Type Number</p> <hr> <p>Adult 11 Boston terrier 4 Boxer 4 Lab 1 Mixed Dog? 1 Ped 5 Pit Bull 1</p> <ul> <li>Marker Distributions</li> </ul> <blockquote> <p>For each field, the "1+", "2+", and "3+" columns positivity are summed to create a new variable Positivity. The mean of the Positivity is computed across all fields for a given sample and marker. The means are shown in the table below.</p> </blockquote> <p>Sample.Name type Grade CD14 CD163 CD3 CD79A IBA1</p> <hr> <p>A1 Adult Low Grade 0.1 0.6 2.9 0.0 14.6 A10 Adult High Grade 0.2 0.4 2.5 0.0 20.6 A11 Adult High Grade 0.1 1.6 0.3 0.0 22.5 A2 Adult Low Grade 0.0 0.6 3.9 0.0 22.5 A3 Adult High Grade 0.8 0.9 0.2 0.0 42.5 A4 Adult High Grade 1.3 7.0 0.7 0.4 34.0 A5 Adult High Grade 1.6 24.7 2.0 0.0 38.9 A6 Adult High Grade 0.1 0.1 0.5 0.0 9.4 A7 Adult High Grade 0.1 5.7 0.3 0.0 31.9 A8 Adult Low Grade 0.2 3.0 1.7 0.2 14.9 A9 Adult High Grade 0.1 0.1 0.2 0.0 18.5 N09-531 Canine High Grade 0.1 0.0 0.3 0.1 34.1 N09-547 Canine High Grade 0.3 20.0 0.7 0.1 28.9 N09-651 Canine High Grade 0.0 15.2 4.6 0.2 28.1 N09-868 Canine High Grade 3.3 12.4 0.7 0.0 23.6 N10-159 Canine Low Grade 8.9 11.8 2.5 0.0 24.4 N10-702 Canine Low Grade 1.9 21.2 0.3 0.0 23.6 N12-166 Canine Low Grade 0.1 5.1 1.0 0.0 40.3 N13-901 Canine High Grade 2.6 0.3 0.5 0.2 6.2 N15-161 Canine High Grade 0.6 2.6 0.7 0.1 10.6 N15-604 Canine High Grade 0.6 9.1 4.9 0.0 30.6 N16-199 Canine High Grade 24.5 4.1 0.1 0.0 3.3 11434 Pediatric High Grade 0.0 57.9 0.5 0.1 37.3 KS18-1269 Pediatric High Grade 0.0 16.7 0.4 0.3 45.6 KS19-607 Pediatric Low Grade 5.2 46.1 0.1 0.1 NA S13-28076 Pediatric High Grade 0.0 11.1 0.1 0.2 76.8 S14-12021 Pediatric High Grade 0.1 23.8 0.0 0.1 54.6</p> <h3 id=fig-5b>Fig 5B<a class=headerlink href=#fig-5b title="Permanent link">&para;</a></h3> <blockquote> <p>The violin plots below show the distribution of percentage positivity. The violins are constructed using all the fields. The points are subject means. The markers CD14, CD3, and CD79A are low across grade and subject type.</p> </blockquote> <div class=highlight><pre><span></span><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
</code></pre></div> <p><img alt src=run_all_files/figure-html/unnamed-chunk-43-1.png><!-- --></p> <div class=highlight><pre><span></span><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
</code></pre></div> <p><img alt src=run_all_files/figure-html/unnamed-chunk-43-2.png><!-- --></p> <div class=highlight><pre><span></span><code>`stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
</code></pre></div> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <h3 id=p-values>p-values<a class=headerlink href=#p-values title="Permanent link">&para;</a></h3> <blockquote> <p>For CD163 and IBA1, mean percentage positivities are compared across sample types (Pediatric, Adult, Canine) within high grade patients. Specifically a Wilcoxon two sample p--value is computed for Pediatric vs. Canine subject means and Pediatric vs. Adult subject means.</p> </blockquote> <p>Marker Group 1 Group 2 p-value</p> <hr> <p>IBA1 Pediatric Canine 0.0040 IBA1 Pediatric Adult 0.0162 CD163 Pediatric Canine 0.0485 CD163 Pediatric Adult 0.0283</p> <hr> <h2 id=fig-5_1>Fig 5<a class=headerlink href=#fig-5_1 title="Permanent link">&para;</a></h2> <blockquote> <p><a href=/methods/S16_expression_immune/#cibersort-based-expression-analysis>CIBERSORT based deconvolution of immune microenvironment</a> in human and canine glioma. </p> </blockquote> <h3 id=required-metadata_3>Required metadata<a class=headerlink href=#required-metadata_3 title="Permanent link">&para;</a></h3> <h3 id=fig-s5>Fig S5<a class=headerlink href=#fig-s5 title="Permanent link">&para;</a></h3> <div class=highlight><pre><span></span><code>quartz_off_screen 
                2 
</code></pre></div> <!-- Source file information --> <hr> <div class=md-source-file> <small> Last update: <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-iso_date">2020-02-08</span> </small> </div> <!-- end default contents --> <!-- Get setting from mkdocs.yml, but allow page-level overrides --> <!-- Inject Disqus into current page --> <p><small>Author: sbamin</small></p> <!-- Giscus --> <h2 id=__comments>Comments</h2> <p>Comments are powered by <span class=twemoji><svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 16 16"><path fill-rule=evenodd d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg></span> <a href=https://github.com/TheJacksonLaboratory/canineglioma/discussions>github discussions</a></p> <!-- Replace with generated snippet --> <script src=https://giscus.app/client.js data-repo=TheJacksonLaboratory/canineglioma data-repo-id="MDEwOlJlcG9zaXRvcnkxOTEyOTczMjg=" data-category=webchat data-category-id=DIC_kwDOC2b3MM4CBGtB data-mapping=pathname data-reactions-enabled=1 data-emit-metadata=0 data-input-position=top data-theme=light data-lang=en crossorigin=anonymous async>
	</script> <!-- Reload on palette change --> <script>
	var palette = __md_get("__palette")
	if (palette && typeof palette.color === "object")
	  if (palette.color.scheme === "slate") {
	    var giscus = document.querySelector("script[src*=giscus]")
	    giscus.setAttribute("data-theme", "dark") 
	  }

	/* Register event handlers after documented loaded */
	document.addEventListener("DOMContentLoaded", function() {
	  var ref = document.querySelector("[data-md-component=palette]")
	  ref.addEventListener("change", function() {
	    var palette = __md_get("__palette")
	    if (palette && typeof palette.color === "object") {
	      var theme = palette.color.scheme === "slate" ? "dark" : "light"

	      /* Instruct Giscus to change theme */
	      var frame = document.querySelector(".giscus-frame")
	      frame.contentWindow.postMessage(
	        { giscus: { setConfig: { theme } } },
	        "https://giscus.app"
	      )
	    }
	  })
	})
	</script> </article> </div> </div> <a href=# class="md-top md-icon" data-md-component=top data-md-state=hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12z"/></svg> Back to top </a> </main> <!-- Footer --> <footer class=md-footer> <!-- Link to previous and/or next page --> <nav class="md-footer__inner md-grid" aria-label=Footer> <!-- Link to previous page --> <a href=../ class="md-footer__link md-footer__link--prev" aria-label="Previous: How-To" rel=prev> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </div> <div class=md-footer__title> <div class=md-ellipsis> <span class=md-footer__direction> Previous </span> How-To </div> </div> </a> <!-- Link to next page --> </nav> <!-- Further information --> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <!-- Copyright and theme information --> <div class=md-copyright> <div class=md-copyright__highlight> &copy; 2010-2022: <a href=https://verhaaklab.com title=Home>Verhaak Lab</a> | <a href=https://verhaaklab.com/privacy/ >Disclosure</a> </div> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> <!-- Social links --> <div class=md-social> <a href=https://verhaaklab.com/contact target=_blank rel=noopener title=verhaaklab.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M464 64H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V112c0-26.51-21.49-48-48-48zm0 48v40.805c-22.422 18.259-58.168 46.651-134.587 106.49-16.841 13.247-50.201 45.072-73.413 44.701-23.208.375-56.579-31.459-73.413-44.701C106.18 199.465 70.425 171.067 48 152.805V112h416zM48 400V214.398c22.914 18.251 55.409 43.862 104.938 82.646 21.857 17.205 60.134 55.186 103.062 54.955 42.717.231 80.509-37.199 103.053-54.947 49.528-38.783 82.032-64.401 104.947-82.653V400H48z"/></svg> </a> <a href=https://twitter.com/roelverhaak target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> <a href=https://github.com/sbamin target=_blank rel=noopener title=github.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg> </a> <a href=https://twitter.com/sbamin target=_blank rel=noopener title=twitter.com class=md-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "../..", "features": ["navigation.tabs", "navigation.instant", "navigation.top", "navigation.indexes", "navigation.sections", "content.code.annotate"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../assets/javascripts/workers/search.0bbba5b5.min.js", "version": {"provider": "mike", "default": "latest"}}</script> <script src=../../assets/javascripts/bundle.649a939e.min.js></script> <script src=../../assets/js/extra.js></script> <script src=../../assets/js/mathjax.js></script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js></script> <script src=https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js></script> <script src=../../assets/js/tables.js></script> </body> </html>